# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR Red Hat, Inc.
# This file is distributed under the same license as the PACKAGE package.
#
# Translators:
# giogio <g.sora@tiscali.it>, 2014
# Martijn Dekker <mcdutchie@hotmail.com>, 2012
# Nik Kalach <nikka@fedoraproject.org>, 2012
# Nik Kalach <nikka@fedoraproject.org>, 2012, 2012-2013, 2013
# Nik Kalach <nik.kalach@inbox.ru>, 2013-2014
# Eduardo Vera <hmcb2080@gmail.com>, 2024.
# eduver <hmcb2080@gmail.com>, 2024.
msgid ""
msgstr ""
"Project-Id-Version: anaconda 21.48.22.158\n"
"Report-Msgid-Bugs-To: anaconda-devel-list@redhat.com\n"
"POT-Creation-Date: 2020-04-15 15:07+0200\n"
"PO-Revision-Date: 2024-12-15 23:25+0000\n"
"Last-Translator: eduver <hmcb2080@gmail.com>\n"
"Language-Team: Interlingua <https://translate.fedoraproject.org/projects/"
"anaconda/rhel-7/ia/>\n"
"Language: ia\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Generator: Weblate 5.8.4\n"

#: pyanaconda/bootloader.py:253
msgid "/boot file system"
msgstr ""

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/bootloader.py:410
#, python-format
msgid ""
"RAID sets that contain '%(desc)s' must have one of the following raid "
"levels: %(raid_level)s."
msgstr ""
"Le insimules de RAID que contine '%(desc)s' debe haber un del sequente "
"nivellos de RAID: %(raid_level)s."

#: pyanaconda/bootloader.py:418
#, python-format
msgid ""
"RAID sets that contain '%(desc)s' must have one of the following metadata "
"versions: %(metadata_versions)s."
msgstr ""
"Le insimules de RAID que contine '%(desc)s' debe haber un del sequente "
"versiones de metadatos: %(metadata_versions)s."

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/bootloader.py:426
#, python-format
msgid ""
"RAID sets that contain '%(desc)s' must have one of the following device "
"types: %(types)s."
msgstr ""
"Le insimules de RAID que contine '%(desc)s' debe haber un del sequente typos "
"de dispositivo: %(types)s."

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/bootloader.py:442
#, python-format
msgid "%(name)s must have one of the following disklabel types: %(types)s."
msgstr ""
"%(name)s debe haber un del sequente typos de etiquetta de disco: %(types)s."

#: pyanaconda/bootloader.py:454
#, python-format
msgid "%(desc)s cannot be of type %(type)s."
msgstr "%(desc)s non pote esser del typo %(type)s."

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/bootloader.py:460
#, python-format
msgid "%(desc)s must be mounted on one of %(mountpoints)s."
msgstr "%(desc)s debe esser montate sur un de %(mountpoints)s."

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/bootloader.py:472
#, python-format
msgid "%(desc)s must be between %(min)d and %(max)d MB in size"
msgstr "Le dimension de %(desc)s debe esser inter %(min)d e %(max)d MB"

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/bootloader.py:478
#, python-format
msgid "%(desc)s must not be smaller than %(min)dMB."
msgstr "%(desc)s non debe esser plus parve que %(min)d MB."

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/bootloader.py:487
#, python-format
msgid "%(desc)s must not be larger than %(max)dMB."
msgstr "%(desc)s non debe esser plus grande que %(max)d MB."

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/bootloader.py:505
#, python-format
msgid "%(desc)s must be within the first %(max_end)s of the disk."
msgstr "%(desc)s debe esser intra le prime %(max_end)s del disco."

#: pyanaconda/bootloader.py:515
#, python-format
msgid "%s must be on a primary partition."
msgstr "%s debe esser sur un partition primari."

#: pyanaconda/bootloader.py:599
msgid ""
"Boot loader stage1 device cannot be on an iSCSI disk which is not configured "
"in iBFT."
msgstr ""

#: pyanaconda/bootloader.py:656 pyanaconda/bootloader.py:767
#, python-format
msgid "%s cannot be on an encrypted block device."
msgstr "%s non pote esser sur un dispositivo de blocos cryptate."

#: pyanaconda/bootloader.py:721
#, python-format
msgid ""
"%(bootloader_stage2_description)s cannot be on an iSCSI disk which is not "
"configured in iBFT."
msgstr ""

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/bootloader.py:726
#, python-format
msgid "%(desc)s cannot be of type %(type)s"
msgstr "%(desc)s non pote esser del typo %(type)s"

#: pyanaconda/bootloader.py:1377
#, python-format
msgid ""
"boot loader stage2 device %(stage2dev)s is on a multi-disk array, but boot "
"loader stage1 device %(stage1dev)s is not. A drive failure in %(stage2dev)s "
"could render the system unbootable."
msgstr ""

#: pyanaconda/bootloader.py:1383
#, python-format
msgid ""
"boot loader stage2 device %(stage2dev)s is on a multi-disk array, but boot "
"loader stage1 device %(stage1dev)s is not part of this array. The stage1 "
"boot loader will only be installed to a single drive."
msgstr ""

#: pyanaconda/bootloader.py:1676
#, python-format
msgid ""
"%(deviceName)s may not have enough space for grub2 to embed core.img when "
"using the %(fsType)s file system on %(deviceType)s"
msgstr ""

#: pyanaconda/constants.py:73
msgid "Start VNC"
msgstr "Lancear VNC"

#: pyanaconda/constants.py:74
msgid "Use text mode"
msgstr "Usar modo textual"

#: pyanaconda/constants.py:141
msgid ""
"Simultaneous Multithreading (SMT) technology can provide performance "
"improvements for certain workloads, but introduces several publicly "
"disclosed security issues. You have the option of disabling SMT, which may "
"impact performance. If you choose to leave SMT enabled, please read https://"
"red.ht/rhel-smt to understand your potential risks and learn about other "
"ways to mitigate these risks."
msgstr ""

#: pyanaconda/constants.py:150
msgid ""
"Simultaneous Multithreading (SMT) may improve performance for certain "
"workloads, but introduces several publicly disclosed security issues. You "
"can disable SMT, which may impact performance. Please read https://red.ht/"
"rhel-smt to understand potential risks and learn about ways to mitigate "
"these risks."
msgstr ""

#: pyanaconda/constants.py:158
#, python-format
msgid "The %(password)s is empty."
msgstr ""

#: pyanaconda/constants.py:159
#, python-format
msgid "The %(passwords)s do not match."
msgstr ""

#: pyanaconda/constants.py:160
#, python-format
msgid "The %(passwords)s you entered were different.  Please try again."
msgstr ""

#: pyanaconda/constants.py:163
#, python-format
msgid "The %(password)s is too short"
msgstr ""

#: pyanaconda/constants.py:164
#, python-format
msgid "The %(password)s you have provided is weak."
msgstr ""

#: pyanaconda/constants.py:165
#, python-format
msgid "The %(password)s you have provided is weak: %(error_message)s."
msgstr ""

#: pyanaconda/constants.py:166
msgid "Press Done again to use anyway."
msgstr ""

#: pyanaconda/constants.py:167
#, python-format
msgid ""
"The %(password)s you have provided contains non-ASCII characters. You may "
"not be able to switch between keyboard layouts when typing it."
msgstr ""

#: pyanaconda/constants.py:169
msgid "You will have to press Done twice to confirm it."
msgstr "Tu debera premer sur Presto duo vices pro confirmar lo."

#: pyanaconda/constants.py:170
msgid "Press Done to continue."
msgstr ""

#: pyanaconda/constants.py:173
msgid "Empty"
msgstr "Vacue"

#: pyanaconda/constants.py:174
msgid "Too short"
msgstr ""

#: pyanaconda/constants.py:175
#: pyanaconda/ui/gui/spokes/lib/passphrase.glade.h:7
msgid "Weak"
msgstr "Insecur"

#: pyanaconda/constants.py:176
msgid "Fair"
msgstr "Passabile"

#: pyanaconda/constants.py:177
msgid "Good"
msgstr "Bon"

#: pyanaconda/constants.py:178
msgid "Strong"
msgstr "Secur"

#: pyanaconda/constants.py:181
msgid "password"
msgstr ""

#: pyanaconda/constants.py:182
msgid "passwords"
msgstr ""

#: pyanaconda/constants.py:185
msgid "passphrase"
msgstr ""

#: pyanaconda/constants.py:186
msgid "passphrases"
msgstr ""

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/errors.py:109
#, python-format
msgid ""
"The following errors occurred with your partitioning:\n"
"\n"
"%(errortxt)s\n"
"\n"
"The installation will now terminate."
msgstr ""
"Le sequente errores occurreva durante partitionamento:\n"
"\n"
"%(errortxt)s\n"
"\n"
"Ora le installation essera terminate."

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/errors.py:115
#, python-format
msgid "An error occurred while resizing the device %s."
msgstr ""
"Un error occurreva durante le cambiamento del dimension del dispositivo %s."

#: pyanaconda/errors.py:124
#, python-format
msgid ""
"There is a problem with your existing storage configuration: %(errortxt)s\n"
"\n"
"You must resolve this matter before the installation can proceed. There is a "
"shell available for use which you can access by pressing ctrl-alt-f1 and "
"then ctrl-b 2.\n"
"\n"
"Once you have resolved the issue you can retry the storage scan. If you do "
"not fix it you will have to exit the installer."
msgstr ""

#: pyanaconda/errors.py:133 pyanaconda/ui/gui/main.glade.h:1
msgid "_Exit Installer"
msgstr "_Sortir le installator"

#: pyanaconda/errors.py:133
msgid "_Retry"
msgstr "_Reprobar"

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/errors.py:140
msgid ""
"An error has occurred - no valid devices were found on which to create new "
"file systems.  Please check your hardware for the cause of this problem."
msgstr ""
"Un error ha occurrite - necun dispositivo valide esseva trovate pro crear "
"nove systemas de files. Examina le hardware pro le causa de iste problema."

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/errors.py:147
#, python-format
msgid ""
"The following file systems for your Linux system were not unmounted "
"cleanly.  Would you like to mount them anyway?\n"
"%s"
msgstr ""
"Le sequente systema de file pro tu systema Linux non esseva dismontate "
"correctemente. Totevia tu volerea montar los?\n"
"%s"

#: pyanaconda/errors.py:158
msgid ""
"There is an entry in your /etc/fstab file that contains an invalid or "
"incorrect file system type:\n"
"\n"
msgstr ""

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/errors.py:164
#, python-format
msgid ""
"The ISO image %s has a size which is not a multiple of 2048 bytes.  This may "
"mean it was corrupted on transfer to this computer.\n"
"\n"
"It is recommended that you exit and abort your installation, but you can "
"choose to continue if you think this is in error. Would you like to continue "
"using this image?"
msgstr ""
"Le imagine ISO %s ha un dimension que non es un multiple de 2048 octettos. "
"Ce pote significar que illo era corrumpite durante le transferentia a iste "
"computator.\n"
"\n"
"Il esrecommendante "

#: pyanaconda/errors.py:178
msgid ""
"The installer has tried to mount the installation image, but cannot find it "
"on the hard drive.\n"
"\n"
"Should I try again to locate the image?"
msgstr ""

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/errors.py:188
#, python-format
msgid "An error occurred mounting the source device %s. Retry?"
msgstr ""
"Un error occurreva durante que on montava le dispositivo fonte %s. Tu essaya "
"de nove?"

#: pyanaconda/errors.py:196
#, python-format
msgid ""
"An error occurred unmounting the disc.  Please make sure you're not "
"accessing %s from the shell on tty2 and then click OK to retry."
msgstr ""

#: pyanaconda/errors.py:204
#, python-format
msgid ""
"The group '%s' is required for this installation. This group does not exist. "
"This is a fatal error and installation will be aborted."
msgstr ""

#: pyanaconda/errors.py:210
#, python-format
msgid ""
"You have specified that the group '%s' should be installed.  This group does "
"not exist.  Would you like to ignore this group and continue with "
"installation?"
msgstr ""

#: pyanaconda/errors.py:215
#, python-format
msgid ""
"You have specified that the group '%s' should be excluded from "
"installation.  This group does not exist.  Would you like to ignore this "
"group and continue with installation?"
msgstr ""

#: pyanaconda/errors.py:227
#, python-format
msgid ""
"The package '%s' is required for this installation. This package does not "
"exist. This is a fatal error and installation will be aborted."
msgstr ""

#: pyanaconda/errors.py:234
#, python-format
msgid ""
"You have specified that the package '%s' should be installed.  This package "
"does not exist.  Would you like to ignore this package and continue with "
"installation?"
msgstr ""

#: pyanaconda/errors.py:245
#, python-format
msgid ""
"There was an error running the kickstart script at line %(lineno)s.  This is "
"a fatal error and installation will be aborted.  The details of this error "
"are:\n"
"\n"
"%(details)s"
msgstr ""

#: pyanaconda/errors.py:253
msgid ""
"The following error occurred while installing.  This is a fatal error and "
"installation will be aborted."
msgstr ""

#: pyanaconda/errors.py:261
msgid ""
"The following software marked for installation has errors.  This is likely "
"caused by an error with\n"
"your installation source."
msgstr ""

#: pyanaconda/errors.py:269
msgid ""
"The following error occurred while installing the boot loader. The system "
"will not be bootable. Would you like to ignore this and continue with "
"installation?"
msgstr ""

#: pyanaconda/errors.py:282
#, python-format
msgid ""
"Installation was stopped due to an error installing the boot loader. The "
"exact error message is:\n"
"\n"
"%s\n"
"\n"
"The installer will now terminate."
msgstr ""

#: pyanaconda/errors.py:290
#, python-format
msgid "Unable to encrypt password: unsupported algorithm %s"
msgstr ""

#: pyanaconda/exception.py:104
#, python-format
msgid ""
"The installation was stopped due to what seems to be a problem with your "
"hardware. The exact error message is:\n"
"\n"
"%s.\n"
"\n"
" The installer will now terminate."
msgstr ""
"Le installation se ha stoppate a causa de un problema possibile con le "
"apparato. Le message exacte del error es:\n"
"\n"
"%s.\n"
"\n"
"Nunc le installator se terminara."

#: pyanaconda/exception.py:108
msgid "Hardware error occurred"
msgstr ""

#: pyanaconda/exception.py:143
#, python-format
msgid ""
"\n"
"The installation was stopped due to an error which occurred while running in "
"non-interactive cmdline mode. Since there cannot be any questions in cmdline "
"mode, edit your kickstart file and retry installation. \n"
"The exact error message is: \n"
"\n"
"%s. \n"
"\n"
"The installer will now terminate."
msgstr ""

#: pyanaconda/exception.py:151
#, python-format
msgid ""
"\n"
"Running in cmdline mode, no interactive debugging allowed.\n"
"The exact error message is: \n"
"\n"
"%s.\n"
"\n"
"The installer will now terminate."
msgstr ""

#: pyanaconda/install.py:83
msgid "Configuring installed system"
msgstr "Configuration del systema installate"

#: pyanaconda/install.py:96
msgid "Writing network configuration"
msgstr "Creation del configuration de rete"

#: pyanaconda/install.py:100
msgid "Creating users"
msgstr "Creation de usatores"

#: pyanaconda/install.py:108
msgid "Configuring addons"
msgstr "Configuration del additiones"

#: pyanaconda/install.py:111
msgid "Generating initramfs"
msgstr "Generation de initramfs"

#: pyanaconda/install.py:115
#, python-format
msgid "Joining realm: %s"
msgstr "Connexion al dominio: %s"

#: pyanaconda/install.py:118
msgid "Running post-installation scripts"
msgstr "Execution de scripts post installation"

#: pyanaconda/install.py:139
msgid "Creating snapshots"
msgstr ""

#: pyanaconda/install.py:178
#, python-format
msgid "Waiting for %s threads to finish"
msgstr "Periodo de attender pro le fin de %s filos"

#: pyanaconda/install.py:184
msgid "Setting up the installation environment"
msgstr "Configuration del ambiente de installation"

#: pyanaconda/install.py:213 pyanaconda/kickstart.py:2485
msgid "Running pre-installation scripts"
msgstr "Execution del scripts de pre-installation"

#: pyanaconda/install.py:221
msgid "Discovering realm to join"
msgstr "Discoperta del dominio a connecter"

#: pyanaconda/install.py:265
msgid "Installing boot loader"
msgstr ""

#: pyanaconda/install.py:268 pyanaconda/packaging/livepayload.py:170
#: pyanaconda/packaging/yumpayload.py:1555
msgid "Performing post-installation setup tasks"
msgstr "Execution de cargas de configuration post installation"

#: pyanaconda/iutil.py:789
msgid "No host url"
msgstr "Nulle url de hospite"

#: pyanaconda/iutil.py:810
msgid "malformed URL, cannot parse it."
msgstr "URL mal formate, non pote analysar lo."

#: pyanaconda/iutil.py:826
msgid "URL has no host component"
msgstr "URL non ha componente de hospite"

#: pyanaconda/kickstart.py:191
#, python-format
msgid "Escrow certificate %s requires the network."
msgstr ""
"Le certificato de deposito fiduciari %s necessita un connexion al rete."

#: pyanaconda/kickstart.py:199
#, python-format
msgid ""
"The following error was encountered while downloading the escrow "
"certificate:\n"
"\n"
"%s"
msgstr ""
"Le sequente error ha essite incontrate durante le discargamento del "
"certificato de deposito fiduciari:\n"
"\n"
"%s"

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/kickstart.py:273
#, python-format
msgid "%s is missing. Cannot setup authentication."
msgstr "%s es mancante. Non pote configurar authentication."

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/kickstart.py:298
#, python-format
msgid "autopart fstype of %s is invalid."
msgstr "autopart fstype de %s es invalide."

#: pyanaconda/kickstart.py:342
msgid "GRUB2 does not support installation to a partition."
msgstr "GRUB2 non supporta installation in un partition."

#: pyanaconda/kickstart.py:435
#, python-format
msgid "More than one match found for given boot drive \"%s\"."
msgstr ""

#: pyanaconda/kickstart.py:440
#, python-format
msgid "Requested boot drive \"%s\" doesn't exist or cannot be used."
msgstr ""

#: pyanaconda/kickstart.py:514
#, python-format
msgid ""
"Btrfs partition \"%(device)s\" has a format of \"%(format)s\", but should "
"have a format of \"Btrfs\"."
msgstr ""

#: pyanaconda/kickstart.py:519
#, python-format
msgid "Tried to use undefined partition \"%s\" in Btrfs volume specification."
msgstr ""

#: pyanaconda/kickstart.py:532
msgid ""
"Btrfs volume defined without any member devices.  Either specify member "
"devices or use --useexisting."
msgstr ""

#: pyanaconda/kickstart.py:541 pyanaconda/kickstart.py:928
#: pyanaconda/kickstart.py:1593
#, python-format
msgid "The mount point \"%s\" is not valid.  It must start with a /."
msgstr ""

#: pyanaconda/kickstart.py:556
#, python-format
msgid "Btrfs volume \"%s\" specified with --useexisting does not exist."
msgstr ""

#: pyanaconda/kickstart.py:640
#, python-format
msgid ""
"Disklabel \"%s\" given in clearpart command is not supported on this "
"platform."
msgstr ""

#: pyanaconda/kickstart.py:652
#, python-format
msgid "Disk \"%s\" given in clearpart command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:665
#, python-format
msgid "Device \"%s\" given in clearpart device list does not exist."
msgstr ""

#: pyanaconda/kickstart.py:692
#, python-format
msgid "NIC \"%s\" given in fcoe command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:752
#, python-format
msgid "%s is missing. Cannot setup firewall."
msgstr "%s es mancante. Impossibile de configurar le parafoco."

#: pyanaconda/kickstart.py:807 pyanaconda/kickstart.py:818
#, python-format
msgid "Disk \"%s\" given in ignoredisk command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:831
#, python-format
msgid ""
"Network interface \"%(nic)s\" required by iSCSI \"%(iscsiTarget)s\" target "
"is not up."
msgstr ""

#: pyanaconda/kickstart.py:841
msgid ""
"iscsi --iface must be specified (binding used) either for all targets or for "
"none"
msgstr ""

#: pyanaconda/kickstart.py:934
#, python-format
msgid ""
"No volume group exists with the name \"%s\".  Specify volume groups before "
"logical volumes."
msgstr ""

#: pyanaconda/kickstart.py:941
msgid "Cache PVs must belong to the same VG as the cached LV"
msgstr ""

#: pyanaconda/kickstart.py:948
#, python-format
msgid ""
"No thin pool exists with the name \"%s\". Specify thin pools before thin "
"volumes."
msgstr ""

#: pyanaconda/kickstart.py:956
msgid "logvol --noformat must also use the --name= option."
msgstr ""

#: pyanaconda/kickstart.py:961 pyanaconda/kickstart.py:1031
#, python-format
msgid "Logical volume \"%s\" given in logvol command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:972 pyanaconda/kickstart.py:981
#: pyanaconda/kickstart.py:1041 pyanaconda/kickstart.py:1385
#: pyanaconda/kickstart.py:1394 pyanaconda/kickstart.py:1477
#, python-format
msgid "Target size \"%(size)s\" for device \"%(device)s\" is invalid."
msgstr ""

#: pyanaconda/kickstart.py:995
#, python-format
msgid ""
"Logical volume name \"%(logvol)s\" is already in use in volume group "
"\"%(volgroup)s\"."
msgstr ""

#: pyanaconda/kickstart.py:1002
#, python-format
msgid ""
"No size given for logical volume \"%s\".  Use one of --useexisting, --size, "
"or --percent."
msgstr ""

#: pyanaconda/kickstart.py:1005
#, python-format
msgid ""
"Logical volume size \"%(logvolSize)s\" must be larger than the volume group "
"extent size of \"%(extentSize)s\"."
msgstr ""

#: pyanaconda/kickstart.py:1009
msgid "Percentage must be between 0 and 100."
msgstr ""

#: pyanaconda/kickstart.py:1020 pyanaconda/kickstart.py:1413
#: pyanaconda/kickstart.py:1646
#, python-format
msgid "The \"%s\" file system type is not supported."
msgstr ""

#: pyanaconda/kickstart.py:1173
#, python-format
msgid "Unknown or invalid device '%s' specified"
msgstr ""

#: pyanaconda/kickstart.py:1178
#, python-format
msgid ""
"Unknown or invalid format '%(format)s' specified for device '%(device)s'"
msgstr ""

#: pyanaconda/kickstart.py:1185
#, python-format
msgid "No format on device '%s'"
msgstr ""

#: pyanaconda/kickstart.py:1223
msgid "nvdimm: nvdimm plugin not available."
msgstr ""

#: pyanaconda/kickstart.py:1229 pyanaconda/kickstart.py:1237
#, python-format
msgid "nvdimm: namespace %s not found."
msgstr ""

#: pyanaconda/kickstart.py:1248
#, python-format
msgid "Disk \"%s\" given in nvdimm command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1256 pyanaconda/kickstart.py:1260
#, python-format
msgid "The %s kickstart command is not currently supported."
msgstr ""

#: pyanaconda/kickstart.py:1285
#, python-format
msgid "No disk found for specified BIOS disk \"%s\"."
msgstr ""

#: pyanaconda/kickstart.py:1320
#, python-format
msgid "RAID partition \"%s\" is defined multiple times."
msgstr ""

#: pyanaconda/kickstart.py:1331 pyanaconda/kickstart.py:1568
#, python-format
msgid "PV partition \"%s\" is defined multiple times."
msgstr ""

#: pyanaconda/kickstart.py:1342 pyanaconda/kickstart.py:1578
#, python-format
msgid "Btrfs partition \"%s\" is defined multiple times."
msgstr ""

#: pyanaconda/kickstart.py:1362
#, python-format
msgid "The size \"%s\" is invalid."
msgstr "Le grandor \"%s\" es invalide."

#: pyanaconda/kickstart.py:1369
msgid "part --noformat must also use the --onpart option."
msgstr ""

#: pyanaconda/kickstart.py:1374 pyanaconda/kickstart.py:1468
#, python-format
msgid "Partition \"%s\" given in part command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1428 pyanaconda/kickstart.py:1442
#, python-format
msgid "Disk \"%s\" given in part command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1431
#, python-format
msgid "Cannot install to unpartitionable device \"%s\"."
msgstr ""

#: pyanaconda/kickstart.py:1438
#, python-format
msgid "Disk \"%s\" in part command is not partitioned."
msgstr ""

#: pyanaconda/kickstart.py:1451
#, python-format
msgid "The maximum size \"%s\" is invalid."
msgstr "Le grandor maxime \"%s\" es invalide."

#: pyanaconda/kickstart.py:1600
msgid "raid --noformat must also use the --device option."
msgstr ""

#: pyanaconda/kickstart.py:1605
#, python-format
msgid "RAID device  \"%s\" given in raid command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1628
#, python-format
msgid ""
"RAID device \"%(device)s\" has a format of \"%(format)s\", but should have a "
"format of \"mdmember\"."
msgstr ""

#: pyanaconda/kickstart.py:1633
#, python-format
msgid "Tried to use undefined partition \"%s\" in RAID specification."
msgstr ""

#: pyanaconda/kickstart.py:1667
#, python-format
msgid "RAID volume \"%s\" specified with --useexisting does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1676
#, python-format
msgid "The RAID volume name \"%s\" is already in use."
msgstr ""

#: pyanaconda/kickstart.py:1965
#, python-format
msgid ""
"Physical volume \"%(device)s\" has a format of \"%(format)s\", but should "
"have a format of \"lvmpv\"."
msgstr ""

#: pyanaconda/kickstart.py:1970
#, python-format
msgid "Tried to use undefined partition \"%s\" in Volume Group specification"
msgstr ""

#: pyanaconda/kickstart.py:1976
#, python-format
msgid ""
"Volume group \"%s\" defined without any physical volumes.  Either specify "
"physical volumes or use --useexisting."
msgstr ""
"gruppo de tomo \"%s\" definite sin alcun tomos physic. O specifica tomos "
"physic o usa --useexisting."

#: pyanaconda/kickstart.py:1986
#, python-format
msgid ""
"Volume group given physical extent size of \"%(extentSize)s\", but must be "
"one of:\n"
"%(validExtentSizes)s."
msgstr ""

#: pyanaconda/kickstart.py:1993
msgid ""
"volgroup --noformat and volgroup --useexisting must also use the --name= "
"option."
msgstr ""
"volgroup --noformat e volgroup --useexisting debe etiam usar le option --"
"name= ."

#: pyanaconda/kickstart.py:1998
#, python-format
msgid "Volume group \"%s\" given in volgroup command does not exist."
msgstr "Le gruppo de tomo \"%s\" date in le commando volgroup non existe."

#: pyanaconda/kickstart.py:2001
#, python-format
msgid "The volume group name \"%s\" is already in use."
msgstr "Le nomine de gruppo de volumine “%s” es ja in uso."

#: pyanaconda/kickstart.py:2124
msgid ""
"Incorrectly specified origin of the snapshot. Use format \"VolGroup/LV_name\""
msgstr ""

#: pyanaconda/kickstart.py:2135
#, python-format
msgid ""
"Snapshot: origin \"%(origin)s\" of snapshot \"%(name)s\" is not a valid thin "
"LV device."
msgstr ""

#: pyanaconda/kickstart.py:2143
#, python-format
msgid "Snapshot %s already exists."
msgstr ""

#: pyanaconda/kickstart.py:2179
msgid ""
"The upgrade kickstart command is no longer supported. Upgrade functionality "
"is provided through fedup."
msgstr ""

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/network.py:110
msgid "Host name cannot be None or an empty string."
msgstr "Nomine de hospite non pote esser None o un catena vacue."

#: pyanaconda/network.py:113
msgid "Host name must be 255 or fewer characters in length."
msgstr ""

#: pyanaconda/network.py:116
msgid ""
"Host names can only contain the characters 'a-z', 'A-Z', '0-9', '-', or '.', "
"parts between periods must contain something and cannot start or end with "
"'-'."
msgstr ""

#: pyanaconda/network.py:1632 pyanaconda/ui/gui/spokes/custom.py:566
#: pyanaconda/ui/gui/spokes/network.py:326
#: pyanaconda/ui/gui/spokes/lib/accordion.py:64
msgid "Unknown"
msgstr "Incognite"

#: pyanaconda/network.py:1636
msgid "Connecting..."
msgstr "Connectente…"

#: pyanaconda/network.py:1638
msgid "Disconnecting..."
msgstr "Disconnectente..."

#: pyanaconda/network.py:1660
#, python-format
msgid "Wired (%(interface_name)s) connected"
msgstr "Interfacie con filo (%(interface_name)s) es connectite"

#: pyanaconda/network.py:1663
#, python-format
msgid "Wireless connected to %(access_point)s"
msgstr "Interfacie sin filo es connectite a %(access_point)s"

#: pyanaconda/network.py:1666
#, python-format
msgid "Bond %(interface_name)s (%(list_of_slaves)s) connected"
msgstr ""
"Interfacie aggregate %(interface_name)s (%(list_of_slaves)s) es connectite"

#: pyanaconda/network.py:1670
#, python-format
msgid "Team %(interface_name)s (%(list_of_slaves)s) connected"
msgstr ""

#: pyanaconda/network.py:1674
#, python-format
msgid "Bridge %(interface_name)s (%(list_of_slaves)s) connected"
msgstr ""

#: pyanaconda/network.py:1684
#, python-format
msgid "VLAN %(interface_name)s (%(parent_device)s, ID %(vlanid)s) connected"
msgstr ""

#: pyanaconda/network.py:1702
#, python-format
msgid "Connected: %(list_of_interface_names)s"
msgstr "Connectite: %(list_of_interface_names)s"

#: pyanaconda/network.py:1705
msgid "Not connected"
msgstr "Non connectite"

#: pyanaconda/network.py:1708 pyanaconda/ui/gui/spokes/network.glade.h:45
msgid "No network devices available"
msgstr "Nulle dispositivos de rete disponibile"

#: pyanaconda/product.py:55
#, python-format
msgid "%(productName)s %(productVersion)s INSTALLATION"
msgstr "INSTALLATION DE %(productName)s %(productVersion)s"

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/product.py:59
#, python-format
msgid "New %(name)s %(version)s Installation"
msgstr "Nove installation de %(name)s %(version)s"

#: pyanaconda/pwpolicy.py:124
#, python-format
msgid "policy name required for %s"
msgstr ""

#: pyanaconda/pwpolicy.py:133
#, python-format
msgid "A %(command)s with the name %(policyName)s has already been defined."
msgstr ""

#: pyanaconda/rescue.py:67
#, python-format
msgid "Run %s to unmount the system when you are finished."
msgstr "Exeque %s pro dismontar le systema al fin."

#: pyanaconda/rescue.py:70
msgid "When finished, please exit from the shell and your system will reboot."
msgstr ""

#: pyanaconda/rescue.py:78
msgid "Unable to find /bin/bash to execute!  Not starting shell."
msgstr ""

#: pyanaconda/rescue.py:120
msgid "Rescue"
msgstr "Salvar"

#: pyanaconda/rescue.py:132
msgid "Continue"
msgstr "Continuar"

#: pyanaconda/rescue.py:132
msgid "Read-only mount"
msgstr ""

#: pyanaconda/rescue.py:132
msgid "Skip to shell"
msgstr ""

#: pyanaconda/rescue.py:146
msgid "Please make a selection from the above:  "
msgstr ""

#: pyanaconda/rescue.py:151
#, python-format
msgid ""
"The rescue environment will now attempt to find your Linux installation and "
"mount it under the directory : %s.  You can then make any changes required "
"to your system.  Choose '1' to proceed with this step.\n"
"You can choose to mount your file systems read-only instead of read-write by "
"choosing '2'.\n"
"If for some reason this process does not work choose '3' to skip directly to "
"a shell.\n"
"\n"
msgstr ""

#: pyanaconda/rescue.py:267
msgid "Root Selection"
msgstr ""

#: pyanaconda/rescue.py:284
msgid "The following installations were discovered on your system.\n"
msgstr ""

#: pyanaconda/rescue.py:296
msgid ""
"Please make your selection from the above list.\n"
"Press 'c' to continue after you have made your selection.  "
msgstr ""

#. TRANSLATORS: 'c' to continue
#. TRANSLATORS:'c' to continue
#. TRANSLATORS: 'c' to continue
#. TRANSLATORS:'c' to continue
#. TRANSLATORS: 'c' to continue
#: pyanaconda/rescue.py:305 pyanaconda/ui/tui/hubs/summary.py:153
#: pyanaconda/ui/tui/hubs/__init__.py:123
#: pyanaconda/ui/tui/hubs/__init__.py:149
#: pyanaconda/ui/tui/simpleline/base.py:462
#: pyanaconda/ui/tui/simpleline/base.py:672
#: pyanaconda/ui/tui/spokes/storage.py:279
#: pyanaconda/ui/tui/spokes/storage.py:620
#: pyanaconda/ui/tui/spokes/storage.py:672
#: pyanaconda/ui/tui/spokes/source.py:482
#: pyanaconda/ui/tui/spokes/software.py:171
msgctxt "TUI|Spoke Navigation"
msgid "c"
msgstr "c"

#: pyanaconda/rescue.py:327
msgid "Rescue Mount"
msgstr ""

#: pyanaconda/rescue.py:349
msgid ""
"Your system had dirty file systems which you chose not to mount.  From a "
"shell, you can fsck and mount your partitions manually.\n"
msgstr ""

#: pyanaconda/rescue.py:358
#, python-format
msgid ""
"Your system has been mounted under %(mountpoint)s.\n"
"\n"
"If you would like to make your system the root environment, run the "
"command:\n"
"\n"
"\tchroot %(mountpoint)s\n"
msgstr ""

#: pyanaconda/rescue.py:397
#, python-format
msgid "Run %s to unmount the system when you are finished.\n"
msgstr ""

#: pyanaconda/rescue.py:399
#, python-format
msgid ""
"An error occurred trying to mount some or all of your system.  Some of it "
"may be mounted under %s\n"
"\n"
msgstr ""

#: pyanaconda/rescue.py:406
msgid "You don't have any Linux partitions.  Rebooting.\n"
msgstr "Necun partitiones de Linux. Le computator essera ora reinitiate.\n"

#: pyanaconda/rescue.py:413
msgid "The system will reboot automatically when you exit from the shell.\n"
msgstr ""

#: pyanaconda/rescue.py:417
#, python-format
msgid "You don't have any Linux partitions. %s\n"
msgstr ""

#: pyanaconda/rescue.py:446
msgid "Please press <return> to get a shell. "
msgstr ""

#: pyanaconda/storage_utils.py:56 pyanaconda/storage_utils.py:88
#: pyanaconda/ui/gui/spokes/custom.glade.h:3
msgid "LVM"
msgstr "LVM"

#: pyanaconda/storage_utils.py:57 pyanaconda/storage_utils.py:89
#: pyanaconda/ui/gui/spokes/custom.glade.h:5
msgid "LVM Thin Provisioning"
msgstr "LVM con allocation fin e dynamic"

#: pyanaconda/storage_utils.py:58 pyanaconda/ui/gui/spokes/custom.glade.h:4
msgid "RAID"
msgstr "RAID"

#: pyanaconda/storage_utils.py:59 pyanaconda/storage_utils.py:86
#: pyanaconda/ui/gui/spokes/custom.glade.h:1
msgid "Standard Partition"
msgstr "Partition normal"

#: pyanaconda/storage_utils.py:60 pyanaconda/storage_utils.py:87
#: pyanaconda/ui/gui/spokes/custom.glade.h:2
msgid "Btrfs"
msgstr ""

#: pyanaconda/storage_utils.py:61 pyanaconda/ui/gui/spokes/lib/resize.glade.h:5
msgid "Disk"
msgstr "Disco"

#: pyanaconda/storage_utils.py:72
msgid ""
"The 'swap' area on your computer is used by the operating\n"
"system when running low on memory."
msgstr ""
"Le area «swap» del computator es usate per le systema\n"
"operative quando illo ha carentia de memoria."

#: pyanaconda/storage_utils.py:74
msgid ""
"The 'boot' area on your computer is where files needed\n"
"to start the operating system are stored."
msgstr ""
"Le area «boot» del computator es un loco ubi reside \n"
"files necessari pro initiar le systema operative."

#: pyanaconda/storage_utils.py:76
msgid ""
"The 'root' area on your computer is where core system\n"
"files and applications are stored."
msgstr ""
"Le area «root» del computator es un loco ubi reside\n"
"files principal e applicationes del systema."

#: pyanaconda/storage_utils.py:78
msgid ""
"The 'home' area on your computer is where all your personal\n"
"data is stored."
msgstr ""
"Le area «home» del computator es un loco ubi reside tote\n"
"le datos personal de usatores."

#: pyanaconda/storage_utils.py:80
msgid ""
"The BIOS boot partition is required to enable booting\n"
"from GPT-partitioned disks on BIOS hardware."
msgstr ""
"Le partition «BIOS boot» se require pro permitter le initio \n"
"de discos partitionate GPT sur apparato con BIOS."

#: pyanaconda/storage_utils.py:82
msgid ""
"The PReP boot partition is required as part of the\n"
"boot loader configuration on some PPC platforms."
msgstr ""

#: pyanaconda/storage_utils.py:159
#, python-format
msgid ""
"Your root partition is less than %(size)s which is usually too small to "
"install %(product)s."
msgstr ""

#: pyanaconda/storage_utils.py:165
#, python-format
msgid ""
"You have not defined a root partition (/), which is required for "
"installation of %s to continue."
msgstr ""
"Tu non ha definite un partition de radice (/) que es necessari pro continuar "
"le installation de %s."

#: pyanaconda/storage_utils.py:178
#, python-format
msgid "Your root partition must be on a device of type: %s."
msgstr ""

#: pyanaconda/storage_utils.py:194
#, python-format
msgid ""
"Your %(mount)s partition is less than %(size)s which is lower than "
"recommended for a normal %(productName)s install."
msgstr ""

#: pyanaconda/storage_utils.py:202
#, python-format
msgid ""
"Your %(mount)s partition is less than %(size)s which is lower than required."
msgstr ""

#: pyanaconda/storage_utils.py:220
#, python-format
msgid ""
"Your %(mount)s partition is too small for %(format)s formatting (allowable "
"size is %(minSize)s to %(maxSize)s)"
msgstr ""

#: pyanaconda/storage_utils.py:226
#, python-format
msgid ""
"Your %(mount)s partition is too large for %(format)s formatting (allowable "
"size is %(minSize)s to %(maxSize)s)"
msgstr ""

#: pyanaconda/storage_utils.py:245
msgid "No valid boot loader target device found. See below for details."
msgstr ""

#: pyanaconda/storage_utils.py:260
msgid "You have not created a bootable partition."
msgstr "Necun partition de boot."

#: pyanaconda/storage_utils.py:295
msgid ""
"Your BIOS-based system needs a special partition to boot from a GPT disk "
"label. To continue, please create a 1MiB 'biosboot' type partition."
msgstr ""

#: pyanaconda/storage_utils.py:316
#, python-format
msgid ""
"You have not specified a swap partition. %(requiredMem)s of memory is "
"required to continue installation without a swap partition, but you only "
"have %(installedMem)s."
msgstr ""

#: pyanaconda/storage_utils.py:322
msgid ""
"You have not specified a swap partition. Although not strictly required in "
"all cases, it will significantly improve performance for most installations."
msgstr ""

#: pyanaconda/storage_utils.py:340
msgid ""
"At least one of your swap devices does not have a UUID, which is common in "
"swap space created using older versions of mkswap. These devices will be "
"referred to by device path in /etc/fstab, which is not ideal since device "
"paths can change under a variety of circumstances. "
msgstr ""
"Al minus un del dispositivos de swap non ha un UUID, que es commun quando le "
"spatio de swap ha essite create per versiones plus vetere de mkswap. Iste "
"dispositivos essera indicate per lor percursos in le file /etc/fstab, que "
"non es ideal proque le percursos de dispositivos pote cambiar se secundo le "
"circumstantias. "

#: pyanaconda/storage_utils.py:361
#, python-format
msgid ""
"This mount point is invalid. The %s directory must be on the / file system."
msgstr ""

#: pyanaconda/storage_utils.py:377
#, python-format
msgid "Your %s must be on a separate partition or LV."
msgstr "La %s debe esser sur un partition separate o LV."

#: pyanaconda/storage_utils.py:394
#, python-format
msgid "The mount point %s must be on a linux file system."
msgstr "Le puncto de montage %s debe esser sur un systema de files de linux."

#: pyanaconda/storage_utils.py:413
#, python-format
msgid "LUKS device %s has no encryption key"
msgstr ""

#: pyanaconda/users.py:234
#, python-format
msgid "User name is reserved for system: %s"
msgstr ""

#: pyanaconda/users.py:237
msgid "User name cannot start with '-' character"
msgstr ""

#: pyanaconda/users.py:246
#, python-format
msgid "User name cannot contain character: '%s'"
msgstr ""

#: pyanaconda/users.py:249
msgid "User name must be shorter than 33 characters"
msgstr ""

#: pyanaconda/vnc.py:76
#, python-format
msgid "%(productName)s %(productVersion)s installation"
msgstr "Installation de %(productName)s %(productVersion)s"

#: pyanaconda/vnc.py:139
#, python-format
msgid "%(productName)s %(productVersion)s installation on host %(name)s"
msgstr ""
"Installation de %(productName)s %(productVersion)s sur le host %(name)s"

#: pyanaconda/vnc.py:158
#, python-format
msgid "Attempting to connect to vnc client on host %s..."
msgstr "Tentativa de connecter al cliente VNC sur le host %s..."

#: pyanaconda/vnc.py:172
msgid "Connected!"
msgstr "Connectite!"

#: pyanaconda/vnc.py:175
msgid "Will try to connect again in 15 seconds..."
msgstr "Reprobara de connecter in 15 secundas..."

#: pyanaconda/vnc.py:182
#, python-format
msgid "Giving up attempting to connect after %d try!\n"
msgid_plural "Giving up attempting to connect after %d tries!\n"
msgstr[0] "Abandono del connexion post %d tentativa!\n"
msgstr[1] "Abandono del connexion post %d tentativas!\n"

#: pyanaconda/vnc.py:190
msgid "Attempting to start vncconfig"
msgstr ""

#: pyanaconda/vnc.py:203
#, python-format
msgid "Please manually connect your vnc client to %s to begin the install."
msgstr ""
"Connecte manualmente le cliente VNC a %s pro comenciar le installation."

#: pyanaconda/vnc.py:205
#, python-format
msgid ""
"Please manually connect your vnc client to <IP ADDRESS>:%s to begin the "
"install. Switch to the shell (Ctrl-B 2) and run 'ip addr' to find the <IP "
"ADDRESS>."
msgstr ""
"Connecte manualmente le cliente VNC a <ADRESSE IP>:%s pro comenciar le "
"installation. Aperi le shell (Ctrl-B 2) e exeque 'ip addr' pro discoperir le "
"<ADRESSE IP>."

#: pyanaconda/vnc.py:210
msgid "Starting VNC..."
msgstr "Initio de VNC..."

#: pyanaconda/vnc.py:247
msgid "The VNC server is now running."
msgstr "Le servitor VNC es nunc active."

#: pyanaconda/vnc.py:251
msgid ""
"\n"
"\n"
"You chose to connect to a listening vncviewer. \n"
"This does not require a password to be set.  If you \n"
"set a password, it will be used in case the connection \n"
"to the vncviewer is unsuccessful\n"
"\n"
msgstr ""
"\n"
"\n"
"Tu ha eligite pro connecter a un vncviewer ascoltante. \n"
"Isto non require le definition de un contrasigno. Si tu \n"
"defini un contrasigno, illo essera usate si le connexion \n"
"al vncviewer non succede.\n"
"\n"

#: pyanaconda/vnc.py:256
msgid ""
"\n"
"\n"
"WARNING!!! VNC server running with NO PASSWORD!\n"
"You can use the vncpassword=<password> boot option\n"
"if you would like to secure the server.\n"
"\n"
msgstr ""
"\n"
"\n"
"ATTENTION: le servitor VNC se es exequente SIN CONTRASIGNO!\n"
"Il es possibile usar le option de boot vncpassword=<contrasigno>\n"
"pro proteger le servitor.\n"
"\n"

#: pyanaconda/vnc.py:260
msgid ""
"\n"
"\n"
"You chose to execute vnc with a password. \n"
"\n"
msgstr ""
"\n"
"\n"
"Tu eligite pro executar VNC con un contrasigno. \n"
"\n"

#: pyanaconda/vnc.py:262
msgid ""
"\n"
"\n"
"Unknown Error.  Aborting. \n"
"\n"
msgstr ""
"\n"
"\n"
"Error incognite. Abandono. \n"
"\n"

#: pyanaconda/vnc.py:284
msgid ""
"VNC password must be six to eight characters long.\n"
"Please enter a new one, or leave blank for no password."
msgstr ""
"Le longitude de contrasigno VNC debe esser de 6 a 8 characteres.\n"
"Introduce un nove contrasigno o lassa lo in blanco pro nulle contrasigno."

#: pyanaconda/packaging/__init__.py:1374
msgid "Failed to set up installation source"
msgstr "Impossibile de configurar le fonte de installation"

#: pyanaconda/packaging/__init__.py:1375
msgid "Error downloading package metadata"
msgstr "Error al discargamento del metadatos de pacchetto"

#: pyanaconda/packaging/__init__.py:1376
msgid "No installation source available"
msgstr ""

#: pyanaconda/packaging/livepayload.py:98
#: pyanaconda/packaging/livepayload.py:119
msgid "Installing software"
msgstr "Installation de software"

#: pyanaconda/packaging/livepayload.py:220
#: pyanaconda/packaging/livepayload.py:229
#, python-format
msgid "Downloading %(url)s (%(pct)d%%)"
msgstr "Discargamento de %(url)s (%(pct)d%%)"

#: pyanaconda/packaging/livepayload.py:358
msgid "Checking image checksum"
msgstr "Verification del summa de controlo del imagine"

#: pyanaconda/packaging/yumpayload.py:1363
#, python-format
msgid "CmdlineError: Missing package: %s"
msgstr "CmdlineError: Pacchetto mancante: %s"

#: pyanaconda/packaging/yumpayload.py:1518
msgid "Starting package installation process"
msgstr "Initio del processo de installation de pacchettos"

#: pyanaconda/packaging/yumpayload.py:1552
msgid "Preparing transaction from installation source"
msgstr "Preparation de transaction ab le fonte de installation"

#: pyanaconda/packaging/yumpayload.py:1553
msgid "Installing"
msgstr "Installation"

#: pyanaconda/packaging/rpmostreepayload.py:163
#, python-format
msgid "Starting pull of %(branchName)s from %(source)s"
msgstr ""

#: pyanaconda/packaging/rpmostreepayload.py:193
#, python-format
msgid "Preparing deployment of %s"
msgstr ""

#: pyanaconda/packaging/rpmostreepayload.py:205
#, python-format
msgid "Deployment starting: %s"
msgstr ""

#: pyanaconda/packaging/rpmostreepayload.py:208
#, python-format
msgid "Deployment complete: %s"
msgstr ""

#: pyanaconda/ui/helpers.py:98 pyanaconda/ui/tui/spokes/storage.py:458
msgid "Checking storage configuration..."
msgstr "Verification del configuration de immagazinage..."

#: pyanaconda/ui/categories/__init__.py:48
msgid "DEFAULT TITLE"
msgstr "TITULO PREDEFINITE"

#: pyanaconda/ui/categories/customization.py:32
msgid "CUSTOMIZATION"
msgstr "PERSONALISATION"

#: pyanaconda/ui/categories/localization.py:31
msgid "LOCALIZATION"
msgstr "LOCALISATION"

#: pyanaconda/ui/categories/software.py:31
msgid "SOFTWARE"
msgstr "SOFTWARE"

#: pyanaconda/ui/categories/system.py:32
#: pyanaconda/ui/gui/spokes/lib/accordion.py:180
msgid "SYSTEM"
msgstr "SYSTEMA"

#: pyanaconda/ui/categories/user_settings.py:32
msgid "USER SETTINGS"
msgstr "PARAMETROS DE CONFIGURATION DE USATOR"

#: pyanaconda/ui/lib/space.py:37
#, python-format
msgid ""
"Not enough space in file systems for the current software selection. An "
"additional %s is needed."
msgstr ""

#: pyanaconda/ui/lib/entropy.py:58
msgid "The system needs more random data entropy"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:67
msgid "Entropy can be increased by typing randomly on keyboard"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:68
#, python-format
msgid ""
"After %d minutes, the installation will continue regardless of the amount of "
"available entropy"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:92
#, python-format
msgid ""
"Available entropy: %(av_entr)s, Required entropy: %(req_entr)s [%(pct)d %%] "
"(%(rem)d %(min)s remaining)"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:95
#: pyanaconda/ui/gui/spokes/lib/entropy_dialog.py:92
msgid "minute"
msgid_plural "minutes"
msgstr[0] ""

#: pyanaconda/ui/lib/entropy.py:102
#, python-format
msgid ""
"Available entropy: %(av_entr)s, Required entropy: %(req_entr)s [%(pct)d %%]"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:107
msgid "Enough entropy gathered, please stop typing."
msgstr ""

#: pyanaconda/ui/lib/entropy.py:110
#, python-format
msgid "Giving up, time (%d minutes) ran out."
msgstr ""

#: pyanaconda/ui/lib/disks.py:113
#, python-format
msgid ""
"You selected disk %(selected)s, which contains devices that also use "
"unselected disk %(unselected)s. You must select or de-select these disks as "
"a set."
msgid_plural ""
"You selected disk %(selected)s, which contains devices that also use "
"unselected disks %(unselected)s. You must select or de-select these disks as "
"a set."
msgstr[0] ""

#: pyanaconda/ui/tui/hubs/summary.py:35
msgid "Installation"
msgstr "Installation"

#: pyanaconda/ui/tui/hubs/summary.py:51
msgid "Starting automated install"
msgstr "Initio de installation automatic"

#. TRANSLATORS: 'b' to begin installation
#: pyanaconda/ui/tui/hubs/summary.py:93
#, python-format
msgid "Enter '%s' to ignore the warning and attempt to install anyway."
msgstr ""

# translation auto-copied from project anaconda, version rhel7-branch,
# document anaconda
#. TRANSLATORS: 'b' to begin installation
#: pyanaconda/ui/tui/hubs/summary.py:95 pyanaconda/ui/tui/hubs/summary.py:122
#: pyanaconda/ui/tui/hubs/summary.py:139
msgctxt "TUI|Spoke Navigation"
msgid "b"
msgstr "b"

#: pyanaconda/ui/tui/hubs/summary.py:104
msgid "The following mandatory spokes are not completed:"
msgstr "Le sequente succursales obligatori non es complite:"

#: pyanaconda/ui/tui/hubs/summary.py:118
#, python-format
msgid ""
"  Please make your choice from above ['%(quit)s' to quit | '%(begin)s' to "
"begin installation |\n"
"  '%(refresh)s' to refresh]: "
msgstr ""

#. TRANSLATORS: 'q' to quit
#: pyanaconda/ui/tui/hubs/summary.py:120 pyanaconda/ui/tui/hubs/__init__.py:147
#: pyanaconda/ui/tui/simpleline/base.py:468
#: pyanaconda/ui/tui/simpleline/base.py:670
msgctxt "TUI|Spoke Navigation"
msgid "q"
msgstr "q"

#. TRANSLATORS: 'r' to refresh
#. TRANSLATORS:'r' to refresh
#. TRANSLATORS: 'r' to refresh
#. TRANSLATORS:'r' to refresh
#: pyanaconda/ui/tui/hubs/summary.py:124 pyanaconda/ui/tui/hubs/__init__.py:151
#: pyanaconda/ui/tui/simpleline/base.py:456
#: pyanaconda/ui/tui/simpleline/base.py:674
msgctxt "TUI|Spoke Navigation"
msgid "r"
msgstr "r"

#: pyanaconda/ui/tui/hubs/summary.py:142 pyanaconda/ui/tui/hubs/__init__.py:126
msgid "Please complete all spokes before continuing"
msgstr "Comple tote le succursales ante de continuar"

#: pyanaconda/ui/tui/hubs/__init__.py:45
msgid "Default HUB title"
msgstr "Titulo de central predefinite"

#: pyanaconda/ui/tui/hubs/__init__.py:143
#, python-format
msgid ""
"  Please make your choice from [ '1' to enter the %(spoke_title)s spoke | "
"'%(quit)s' to quit |\n"
"  '%(continue)s' to continue | '%(refresh)s' to refresh]: "
msgstr ""

#: pyanaconda/ui/tui/tuiobject.py:30 pyanaconda/ui/gui/main.glade.h:2
msgid "Error"
msgstr "Error"

#: pyanaconda/ui/tui/tuiobject.py:51
msgid "Press enter to exit."
msgstr "Preme Enter pro sortir."

#: pyanaconda/ui/tui/tuiobject.py:64 pyanaconda/ui/tui/spokes/password.py:72
#: pyanaconda/ui/gui/spokes/password.glade.h:4
#: pyanaconda/ui/gui/spokes/user.glade.h:8
msgid "Password"
msgstr "Contrasigno"

#: pyanaconda/ui/tui/tuiobject.py:85
msgid "Passphrase: "
msgstr ""

#: pyanaconda/ui/tui/tuiobject.py:111
msgid "Question"
msgstr "Question"

#: pyanaconda/ui/tui/tuiobject.py:134
#, python-format
msgid "Please respond '%(yes)s' or '%(no)s': "
msgstr ""

# translation auto-copied from project anaconda, version f22-branch, document
# anaconda
#. TRANSLATORS: 'yes' as positive reply
#: pyanaconda/ui/tui/tuiobject.py:136 pyanaconda/ui/tui/tuiobject.py:143
msgctxt "TUI|Spoke Navigation"
msgid "yes"
msgstr "si"

# translation auto-copied from project anaconda, version f22-branch, document
# anaconda
#. TRANSLATORS: 'no' as negative reply
#: pyanaconda/ui/tui/tuiobject.py:138 pyanaconda/ui/tui/tuiobject.py:149
msgctxt "TUI|Spoke Navigation"
msgid "no"
msgstr "no"

#: pyanaconda/ui/tui/simpleline/base.py:92
msgid "Do you really want to quit?"
msgstr "Quitar realmente?"

#: pyanaconda/ui/tui/simpleline/base.py:617 anaconda:538
msgid "Press ENTER to continue"
msgstr "Pressa sur INTRO pro continuar"

#: pyanaconda/ui/tui/simpleline/base.py:668
#, python-format
msgid ""
"  Please make your choice from above ['%(quit)s' to quit | '%(continue)s' to "
"continue |\n"
"  '%(refresh)s' to refresh]: "
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:40
msgid "VNC"
msgstr "VNC"

#: pyanaconda/ui/tui/spokes/askvnc.py:57
msgid ""
"X was unable to start on your machine.  Would you like to start VNC to "
"connect to this computer from another computer and perform a graphical "
"installation or continue with a text mode installation?"
msgstr ""
"Impossibile de initiar X in iste machina. Velle tu initiar VNC pro connecter "
"a iste computator de un altere computator e exequer un installation graphic, "
"o continuar con le installation in modo textual?"

#: pyanaconda/ui/tui/spokes/askvnc.py:106
msgid "q"
msgstr "q"

#: pyanaconda/ui/tui/spokes/askvnc.py:123
msgid "VNC Password"
msgstr "Contrasigno VNC"

#: pyanaconda/ui/tui/spokes/askvnc.py:131
msgid ""
"Please provide VNC password (must be six to eight characters long).\n"
"You will have to type it twice. Leave blank for no password"
msgstr ""
"Forni un contrasigno VNC (illo debe esser de 6 a 8 characteres).\n"
"Tu debera facer lo duo vices. Lassa lo in blanco pro le uso sin contrasigno"

#: pyanaconda/ui/tui/spokes/askvnc.py:150
#: pyanaconda/ui/gui/spokes/network.py:1426
msgid "Password: "
msgstr "Contrasigno: "

#: pyanaconda/ui/tui/spokes/askvnc.py:151
msgid "Password (confirm): "
msgstr "Contrasigno (confirma): "

#: pyanaconda/ui/tui/spokes/askvnc.py:154
msgid "Passwords do not match!"
msgstr "Le contrasignos non correspondente!"

#: pyanaconda/ui/tui/spokes/askvnc.py:157
msgid "The password must be at least six characters long."
msgstr "Le contrasigno debe haber al minus sex characteres de longitude."

#: pyanaconda/ui/tui/spokes/askvnc.py:161
msgid "The password cannot be more than eight characters long."
msgstr "Le contrasigno non pote esser plus longe que 8 characteres."

#: pyanaconda/ui/tui/spokes/kernel_warning.py:36
msgid "Warning: Processor has Simultaneous Multithreading (SMT) enabled"
msgstr ""

#: pyanaconda/ui/tui/spokes/langsupport.py:39
msgid "Language settings"
msgstr ""

#: pyanaconda/ui/tui/spokes/langsupport.py:73
msgid "Language is not set."
msgstr ""

#: pyanaconda/ui/tui/spokes/langsupport.py:83
msgid "Available locales"
msgstr ""

#: pyanaconda/ui/tui/spokes/langsupport.py:86
msgid "Available languages"
msgstr ""

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#. TRANSLATORS: 'b' to go back
#: pyanaconda/ui/tui/spokes/langsupport.py:120
#: pyanaconda/ui/tui/spokes/langsupport.py:130
msgctxt "TUI|Spoke Navigation|Language Support"
msgid "b"
msgstr "b"

#: pyanaconda/ui/tui/spokes/langsupport.py:128
#, python-format
msgid ""
"Please select language support to install.\n"
"[%(back)s to return to language list, %(continue)s to continue, %(quit)s to "
"quit]: "
msgstr ""

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#. TRANSLATORS:'c' to continue
#: pyanaconda/ui/tui/spokes/langsupport.py:132
msgctxt "TUI|Spoke Navigation|Language Support"
msgid "c"
msgstr "c"

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#. TRANSLATORS:'q' to quit
#: pyanaconda/ui/tui/spokes/langsupport.py:134
msgctxt "TUI|Spoke Navigation|Language Support"
msgid "q"
msgstr "q"

#: pyanaconda/ui/tui/spokes/network.py:48
msgid "Network configuration"
msgstr "Configuration de rete"

#: pyanaconda/ui/tui/spokes/network.py:123
#, python-format
msgid "Wired (%(interface_name)s) disconnected\n"
msgstr "Interfacie con filo (%(interface_name)s) es disconnectite\n"

#: pyanaconda/ui/tui/spokes/network.py:128
#, python-format
msgid "Wired (%(interface_name)s) connected\n"
msgstr "Interfacie con filo (%(interface_name)s) es connectite\n"

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/ui/tui/spokes/network.py:140
#, python-format
msgid " IPv4 Address: %(addr)s Netmask: %(netmask)s Gateway: %(gateway)s\n"
msgstr ""
" Adresse IPv4: %(addr)s Masca de rete: %(netmask)s Passarella: %(gateway)s\n"

#: pyanaconda/ui/tui/spokes/network.py:142
#, python-format
msgid " DNS: %s\n"
msgstr " DNS: %s\n"

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/ui/tui/spokes/network.py:149
#, python-format
msgid " IPv6 Address: %(addr)s/%(prefix)d\n"
msgstr " Adresse IPv6: %(addr)s/%(prefix)d\n"

#: pyanaconda/ui/tui/spokes/network.py:163
#, python-format
msgid "Host name: %s\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:165
#, python-format
msgid "Current host name: %s\n"
msgstr ""

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/ui/tui/spokes/network.py:177
msgid "Set host name"
msgstr "Fixa nomine de hospite"

#: pyanaconda/ui/tui/spokes/network.py:179
#, python-format
msgid "Configure device %s"
msgstr "Configurar le dispositivo %s"

#: pyanaconda/ui/tui/spokes/network.py:198
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:6
#: pyanaconda/ui/gui/spokes/network.glade.h:51
msgid "Host name"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:232
msgid "Configuration not saved: netmask missing in static configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:250
msgid "Can't apply configuration, device activation failed."
msgstr ""
"Impossibile de applicar le configuration, le activation de dispositivo ha "
"fallite. "

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/ui/tui/spokes/network.py:292
#: pyanaconda/ui/gui/spokes/network.py:1597
#: pyanaconda/ui/gui/spokes/network.py:1616
#: pyanaconda/ui/gui/spokes/network.py:1686
#: pyanaconda/ui/gui/spokes/network.py:1702
#, python-format
msgid "Host name is not valid: %s"
msgstr "Nomine de hospite invalide; %s"

#: pyanaconda/ui/tui/spokes/network.py:321
msgid "Device configuration"
msgstr "Configuration de dispositivo"

#: pyanaconda/ui/tui/spokes/network.py:325
#, python-format
msgid "IPv4 address or %s for DHCP"
msgstr "Adresse IPv4 o %s pro DHCP"

#: pyanaconda/ui/tui/spokes/network.py:327
msgid "IPv4 netmask"
msgstr "Masca de rete IPv4"

#: pyanaconda/ui/tui/spokes/network.py:328
msgid "IPv4 gateway"
msgstr "Passarella IPv4"

#: pyanaconda/ui/tui/spokes/network.py:329
#, python-format
msgid ""
"IPv6 address[/prefix] or %(auto)s for automatic, %(dhcp)s for DHCP, "
"%(ignore)s to turn off"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:332
msgid "IPv6 default gateway"
msgstr "Passarella predefinite IPv6"

#: pyanaconda/ui/tui/spokes/network.py:333
msgid "Nameservers (comma separated)"
msgstr "Servitores de nomines (separate per commas)"

#: pyanaconda/ui/tui/spokes/network.py:339
msgid "Connect automatically after reboot"
msgstr "Connecter automaticamente post reinitio"

#: pyanaconda/ui/tui/spokes/network.py:340
msgid "Apply configuration in installer"
msgstr "Applicar le configuration in le installator"

#: pyanaconda/ui/tui/spokes/network.py:363
#, python-format
msgid "Configuring device %s."
msgstr "Configuration del dispositivo %s."

#: pyanaconda/ui/tui/spokes/network.py:368
msgid "Bad format of the IP address"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:375
msgid "Bad format of the netmask"
msgstr ""

#: pyanaconda/ui/tui/spokes/password.py:32
msgid "Root password"
msgstr "Contrasigno del superusator"

#: pyanaconda/ui/tui/spokes/password.py:57
msgid "Password is set."
msgstr "Le contrasigno es definite."

#: pyanaconda/ui/tui/spokes/password.py:59
msgid "Root account is disabled."
msgstr "Le conto de superusator es disactivate."

#: pyanaconda/ui/tui/spokes/password.py:61
msgid "Password is not set."
msgstr "Le contrasigno non es definite."

#: pyanaconda/ui/tui/spokes/password.py:66
msgid "Please select new root password. You will have to type it twice."
msgstr ""
"Selige un nove contrasigno de superusator. Tu debera introducer lo duo vices."

#: pyanaconda/ui/tui/spokes/storage.py:56
msgid "Use All Space"
msgstr "Usar tote le spatio"

#: pyanaconda/ui/tui/spokes/storage.py:57
msgid "Replace Existing Linux system(s)"
msgstr "Reimplaciar le systema(s) Linux existente"

#: pyanaconda/ui/tui/spokes/storage.py:58
msgid "Use Free Space"
msgstr "Usar le spatio libere"

#: pyanaconda/ui/tui/spokes/storage.py:68
msgid "Installation Destination"
msgstr "Destination de installation"

#: pyanaconda/ui/tui/spokes/storage.py:128
#: pyanaconda/ui/gui/spokes/storage.py:439
msgid "No disks selected"
msgstr "Nulle discos seligite"

#: pyanaconda/ui/tui/spokes/storage.py:138
#: pyanaconda/ui/gui/spokes/storage.py:451
msgid "Error checking storage configuration"
msgstr "Error al verification del configuration del immagazinage"

#: pyanaconda/ui/tui/spokes/storage.py:140
#: pyanaconda/ui/gui/spokes/storage.py:453
msgid "Warning checking storage configuration"
msgstr "Aviso al controlo del configuration de immagazinage"

#: pyanaconda/ui/tui/spokes/storage.py:143
#: pyanaconda/ui/gui/spokes/storage.py:455
msgid "Automatic partitioning selected"
msgstr "Partitionamento automatic seligite"

#: pyanaconda/ui/tui/spokes/storage.py:145
#: pyanaconda/ui/gui/spokes/storage.py:457
msgid "Custom partitioning selected"
msgstr "Partitionamento personalisate seligite"

#: pyanaconda/ui/tui/spokes/storage.py:181
#: pyanaconda/ui/gui/spokes/storage.py:717
msgid ""
"No disks detected.  Please shut down the computer, connect at least one "
"disk, and restart to complete installation."
msgstr ""
"Nulle discos discoperite. Extingue le computator, connecte al minus un disco "
"e reinitia pro ultimar le installation."

#: pyanaconda/ui/tui/spokes/storage.py:183
#: pyanaconda/ui/gui/spokes/storage.py:721
msgid "No disks selected; please select at least one disk to install to."
msgstr "Nulle discos seligite; selige al minus un disco pro le installation."

#: pyanaconda/ui/tui/spokes/storage.py:198
#: pyanaconda/ui/tui/spokes/source.py:397
#: pyanaconda/ui/gui/spokes/source.py:697
#: pyanaconda/ui/gui/spokes/storage.py:665
msgid "Probing storage..."
msgstr "Probation de immagazinage..."

#: pyanaconda/ui/tui/spokes/storage.py:219
msgid "Select all"
msgstr "Seliger toto"

#: pyanaconda/ui/tui/spokes/storage.py:346
msgid ""
"The following unformatted or LDL DASDs have been detected on your system. "
"You can choose to format them now with dasdfmt or cancel to leave them "
"unformatted. Unformatted DASDs cannot be used during installation.\n"
"\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:351
msgid ""
"Warning: All storage changes made using the installer will be lost when you "
"choose to format.\n"
"\n"
"Proceed to run dasdfmt?\n"
msgstr ""
"Aviso. Omne modificationes de immagazinage facite durante que on usava le "
"installator essera perdite quando tu selige formatar.\n"
"\n"
"Tu procede con le execution de dasdfmt?\n"

#: pyanaconda/ui/tui/spokes/storage.py:369
#: pyanaconda/ui/gui/spokes/lib/dasdfmt.py:94
#, python-format
msgid "Formatting /dev/%s. This may take a moment."
msgstr "Formatante /dev/%s. Isto poterea prender un momento."

#: pyanaconda/ui/tui/spokes/storage.py:437
msgid "Generating updated storage configuration"
msgstr "Generation del configuration de immagazinage actualisate"

#: pyanaconda/ui/tui/spokes/storage.py:442
#: pyanaconda/ui/tui/spokes/storage.py:454
#, python-format
msgid "storage configuration failed: %s"
msgstr "insuccesso del configuration de immagazinage: %s"

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/ui/tui/spokes/storage.py:501
#: pyanaconda/ui/tui/spokes/storage.py:549
msgid "Passphrase"
msgstr "Phrase de contrasigno"

#: pyanaconda/ui/tui/spokes/storage.py:543
msgid "Please provide LUKS passphrase. You will have to type it twice."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:570
msgid "Autopartitioning Options"
msgstr "Optiones de autopartitionamento"

#: pyanaconda/ui/tui/spokes/storage.py:597
msgid ""
"Installation requires partitioning of your hard drive. Select what space to "
"use for the install target."
msgstr ""
"Le installation necessita le partitionamento del disco. Selige qual spatio "
"utilisar como le scopo del installation."

#: pyanaconda/ui/tui/spokes/storage.py:637
msgid "Partition Scheme Options"
msgstr "Optiones del schema de partitiones"

#: pyanaconda/ui/tui/spokes/storage.py:661
msgid "Select a partition scheme configuration."
msgstr "Selige un configuration del schema de partitiones."

#: pyanaconda/ui/tui/spokes/time_spoke.py:41
msgid "status: working"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:42
msgid "status: not working"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:43
msgid "checking status"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:53
msgid "Time settings"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:189
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:542
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:548
#, python-format
msgid "%s timezone"
msgstr "Fuso horari %s"

#: pyanaconda/ui/tui/spokes/time_spoke.py:191
msgid "Timezone is not set."
msgstr "Fuso horari non es definite."

#: pyanaconda/ui/tui/spokes/time_spoke.py:201
msgid "not set"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:205
#, python-format
msgid "Timezone: %s\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:211
#: pyanaconda/ui/tui/spokes/time_spoke.py:394
msgid "NTP servers:"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:216
msgid "not configured"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:227
msgid "Change timezone"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:229
msgid "Set timezone"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:231
msgid "Configure NTP servers"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:276
msgid "Timezone settings"
msgstr "Parametros de fuso horari"

#: pyanaconda/ui/tui/spokes/time_spoke.py:303
#, python-format
msgid "Available timezones in region %s"
msgstr "Fusos horari disponibile in le region %s"

#: pyanaconda/ui/tui/spokes/time_spoke.py:306
msgid "Available regions"
msgstr "Regiones disponibile"

#. TRANSLATORS: 'b' to go back
#: pyanaconda/ui/tui/spokes/time_spoke.py:345
msgctxt "TUI|Spoke Navigation|Timezone Settings"
msgid "b"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:369
#, python-format
msgid ""
"Please select the timezone.\n"
"Use numbers or type names directly [%(back)s to region list, %(quit)s to "
"quit]: "
msgstr ""

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#. TRANSLATORS: 'b' to go back
#: pyanaconda/ui/tui/spokes/time_spoke.py:371
msgctxt "TUI|Spoke Navigation|Time Settings"
msgid "b"
msgstr "b"

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#. TRANSLATORS:'q' to quit
#: pyanaconda/ui/tui/spokes/time_spoke.py:373
msgctxt "TUI|Spoke Navigation|Time Settings"
msgid "q"
msgstr "q"

#: pyanaconda/ui/tui/spokes/time_spoke.py:381
msgid "NTP configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:399
msgid "no NTP servers have been configured"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:408
msgid "Add NTP server"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:411
msgid "Remove NTP server"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:454
msgid "Add NTP server address"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:473
msgid "Enter an NTP server address and press enter\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:492
msgid "Select an NTP server to remove"
msgstr ""

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/ui/tui/spokes/shell_spoke.py:33
msgid "Shell"
msgstr "Shell"

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/ui/tui/spokes/shell_spoke.py:48
msgid "Start shell"
msgstr "Initia shell"

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/ui/tui/spokes/shell_spoke.py:57
msgid "Exit the shell to continue"
msgstr "Exi le shell pro continuar"

#: pyanaconda/ui/tui/spokes/source.py:54
msgid "Installation source"
msgstr "Fonte de installation"

#: pyanaconda/ui/tui/spokes/source.py:57 pyanaconda/ui/tui/spokes/source.py:128
#: pyanaconda/ui/gui/spokes/source.py:581
#: pyanaconda/ui/gui/spokes/source.glade.h:25
msgid "Closest mirror"
msgstr "Speculo le plus vicin"

#: pyanaconda/ui/tui/spokes/source.py:118
#: pyanaconda/ui/gui/spokes/source.py:571
#, python-format
msgid "NFS server %s"
msgstr "Servitor NFS %s"

#: pyanaconda/ui/tui/spokes/source.py:120
#: pyanaconda/ui/gui/spokes/source.py:573
msgid "Local media"
msgstr "Medio local"

#: pyanaconda/ui/tui/spokes/source.py:122
#: pyanaconda/ui/gui/spokes/source.py:575
msgid "Local media via SE/HMC"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:125
#: pyanaconda/ui/tui/spokes/source.py:139
#: pyanaconda/ui/gui/spokes/source.py:567
msgid "Error setting up software source"
msgstr "Error al configurar un fonte de software"

#: pyanaconda/ui/tui/spokes/source.py:130
#: pyanaconda/ui/tui/spokes/software.py:104
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:550
#: pyanaconda/ui/gui/spokes/software.py:294
#: pyanaconda/ui/gui/spokes/software.py:302
#: pyanaconda/ui/gui/spokes/source.py:583
msgid "Nothing selected"
msgstr "Nihil seligite"

#: pyanaconda/ui/tui/spokes/source.py:141
#: pyanaconda/ui/tui/spokes/software.py:92
msgid "Processing..."
msgstr "In processo…"

#: pyanaconda/ui/tui/spokes/source.py:159
#: pyanaconda/ui/gui/spokes/source.py:914
msgid ""
"The installation source is in use by the installer and cannot be changed."
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:163
msgid "CD/DVD"
msgstr "CD/DVD"

#: pyanaconda/ui/tui/spokes/source.py:163
msgid "local ISO file"
msgstr "file ISO local"

#: pyanaconda/ui/tui/spokes/source.py:163
msgid "Network"
msgstr "Rete"

#: pyanaconda/ui/tui/spokes/source.py:166
msgid "SE/HMC"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:171
msgid "Choose an installation source type."
msgstr "Selige un typo del fonte de installation."

#: pyanaconda/ui/tui/spokes/source.py:274
#: pyanaconda/ui/tui/spokes/source.py:312
msgid "Specify Repo Options"
msgstr "Specificar optiones de repositorio"

#: pyanaconda/ui/tui/spokes/source.py:278
msgid "Repo URL"
msgstr "URL de repositorio"

#: pyanaconda/ui/tui/spokes/source.py:316
msgid "<server>:/<path>"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:317
msgid "NFS mount options"
msgstr "Optiones de montage NFS"

#: pyanaconda/ui/tui/spokes/source.py:359
msgid "Select device containing the ISO file"
msgstr "Seliger un dispositivo continente le file ISO"

#: pyanaconda/ui/tui/spokes/source.py:418
msgid "No mountable devices found"
msgstr "Nulle dispositivos trovate que es appropriate pro montage"

#: pyanaconda/ui/tui/spokes/source.py:446
msgid "Select an ISO to use as install source"
msgstr "Seliger un ISO pro usar como un fonte de installation"

#: pyanaconda/ui/tui/spokes/source.py:475
msgid "No *.iso files found in device root folder"
msgstr "Nulle files *.iso trovate in le directorio de radice del dispositivo"

#: pyanaconda/ui/tui/spokes/software.py:39
msgid "Software selection"
msgstr "Selection de software"

#: pyanaconda/ui/tui/spokes/software.py:90
#: pyanaconda/ui/gui/spokes/software.py:275
msgid "Error checking software selection"
msgstr "Error al verification del selection del programmas"

#: pyanaconda/ui/tui/spokes/software.py:94
#: pyanaconda/ui/gui/spokes/software.py:278
msgid "Installation source not set up"
msgstr "Le fonte de installation non es configurate"

#: pyanaconda/ui/tui/spokes/software.py:96
#: pyanaconda/ui/gui/spokes/software.py:281
msgid "Source changed - please verify"
msgstr "Fonte cambiate - verifica"

#: pyanaconda/ui/tui/spokes/software.py:103
#: pyanaconda/ui/gui/spokes/software.py:288
msgid "Custom software selected"
msgstr "Programma personalisate seligite"

#: pyanaconda/ui/tui/spokes/software.py:128
msgid "Installation source needs to be set up first."
msgstr "Le fonte de installation debe esser configurate in prime loco."

#: pyanaconda/ui/tui/spokes/software.py:138
#: pyanaconda/ui/tui/spokes/software.py:148
msgid "Base environment"
msgstr "Ambiente basic"

#: pyanaconda/ui/tui/spokes/software.py:232
msgid "Checking software selection"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:37
msgid "User creation"
msgstr "Creation de usator"

#: pyanaconda/ui/tui/spokes/user.py:124 pyanaconda/ui/gui/spokes/user.py:377
msgid "No user will be created"
msgstr "Nulle usator essera create"

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/ui/tui/spokes/user.py:126
msgid "You must set a password"
msgstr "Tu debe fixar un contrasigno"

#: pyanaconda/ui/tui/spokes/user.py:128 pyanaconda/ui/gui/spokes/user.py:379
#, python-format
msgid "Administrator %s will be created"
msgstr "Administrator %s essera create"

#: pyanaconda/ui/tui/spokes/user.py:130 pyanaconda/ui/gui/spokes/user.py:381
#, python-format
msgid "User %s will be created"
msgstr "Usator %s essera create"

#: pyanaconda/ui/tui/spokes/user.py:140
msgid "Full name can't contain the ':' character"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:145
msgid ""
"Either a group name in the group list is invalid or groups are not separated "
"by a comma"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:155
#, python-format
msgid ""
"Invalid user name: %(name)s.\n"
"%(error_message)s"
msgstr ""

#: pyanaconda/ui/tui/spokes/warnings.py:36
msgid "Warnings"
msgstr "Advertimentos"

#: pyanaconda/ui/tui/spokes/warnings.py:45
#: pyanaconda/ui/gui/spokes/welcome.glade.h:21
msgid ""
"This hardware (or a combination thereof) is not supported by Red Hat.  For "
"more information on supported hardware, please refer to http://www.redhat."
"com/hardware."
msgstr ""
"Ce hardware (o su combination) non es supportate per Red Hat. Pro plus de "
"information super le hardware supportate, refere se a http://www.redhat.com/"
"hardware."

#: pyanaconda/ui/tui/spokes/progress.py:37
msgid "Progress"
msgstr "Progresso"

#: pyanaconda/ui/tui/spokes/progress.py:141
#, python-format
msgid ""
"\tUse of this product is subject to the license agreement found at %s\n"
"\n"
"\tInstallation complete.  Press return to quit"
msgstr ""
"\tLe usage de iste producto es un subjecto del accordo al licentia que es "
"disponibile a %s\n"
"\n"
"\tLe installation es complete.  Preme sur retorno pro quitar."

#: pyanaconda/ui/tui/spokes/__init__.py:50
msgid "Default spoke title"
msgstr "Titulo de succursal predefinite"

#: pyanaconda/ui/tui/spokes/__init__.py:62
msgid "testing status..."
msgstr "verification del stato..."

#: pyanaconda/ui/tui/spokes/__init__.py:107
msgid "New value"
msgstr "Nove valor"

#: pyanaconda/ui/tui/spokes/__init__.py:145
#, python-format
msgid "%s: "
msgstr "%s: "

#: pyanaconda/ui/tui/spokes/__init__.py:146
#, python-format
msgid "%s (confirm): "
msgstr "%s (confirmar): "

#: pyanaconda/ui/tui/spokes/__init__.py:169
msgid ""
"\n"
"Would you like to use it anyway?"
msgstr ""
"\n"
"Vole tu usar lo comocunque?"

#: pyanaconda/ui/tui/spokes/__init__.py:193
#, python-format
msgid "Enter new value for '%s' and press enter\n"
msgstr "Introduce le nove valor de '%s' e preme sur Enter\n"

#: pyanaconda/ui/tui/spokes/__init__.py:212
msgid "You have provided an invalid value\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:325
msgid "Password set."
msgstr "Le contrasigno es definite."

#: pyanaconda/ui/gui/__init__.py:391 widgets/src/BaseWindow.c:157
msgid "Help!"
msgstr ""

#: pyanaconda/ui/gui/__init__.py:873 widgets/src/StandaloneWindow.c:49
msgid "_Quit"
msgstr "_Quitar"

#: pyanaconda/ui/gui/__init__.py:889
msgid "_No"
msgstr "_No"

#: pyanaconda/ui/gui/__init__.py:889
msgid "_Yes"
msgstr "_Si"

#: pyanaconda/ui/gui/hubs/__init__.py:256
msgid ""
"Please complete items marked with this icon before continuing to the next "
"step."
msgstr ""
"Completa le punctos marcate per iste icona ante de continuar con le sequente "
"passo."

#: pyanaconda/ui/gui/hubs/progress.py:116
msgid "Complete!"
msgstr "Complete!"

#: pyanaconda/ui/gui/hubs/progress.py:144
#, python-format
msgid "Use of this product is subject to the license agreement found at %s"
msgstr ""
"Le usage de iste producto es un subjecto del accordo al licentia que es "
"disponibile a %s"

#: pyanaconda/ui/gui/hubs/progress.py:220
#, python-format
msgid ""
"%s is now successfully installed on your system and ready for you to use!  "
"When you are ready, reboot your system to start using it!"
msgstr ""
"%s es ora installate con successo sur le systema e preste al uso! Procede e "
"reinitia le systema pro comenciar a usar lo!"

#: pyanaconda/ui/gui/hubs/progress.py:223
msgctxt "GUI|Progress"
msgid "_Quit"
msgstr "_Quitar"

#: pyanaconda/ui/gui/hubs/progress.py:231
#, python-format
msgid ""
"%s is now successfully installed, but some configuration still needs to be "
"done.\n"
"Finish it and then click the Finish configuration button please."
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.py:236
#, python-format
msgid ""
"%s is now successfully installed and ready for you to use!\n"
"Go ahead and reboot to start using it!"
msgstr ""

#: pyanaconda/ui/gui/hubs/summary.py:81
msgid ""
"Warning: Processor has Simultaneous Multithreading (SMT) enabled.  <a href="
"\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/hubs/summary.py:91
msgid ""
"The following warnings were encountered when checking your kernel "
"configuration. These are not fatal, but you may wish to make changes to your "
"kernel config."
msgstr ""
"Le sequente avisos se ha incontrate durante le controlo del configuration "
"del nucleo. Istos non es fatal, ma tu pote voler modificar le configuration "
"de nucleo."

#: pyanaconda/ui/gui/hubs/summary.py:99
msgctxt "GUI|Summary|Warning Dialog"
msgid "_OK"
msgstr "_OK"

#: pyanaconda/ui/gui/spokes/custom.py:114
#, python-format
msgid "Create a new %(container_type)s ..."
msgstr "Crear un nove %(container_type)s ..."

#: pyanaconda/ui/gui/spokes/custom.py:115
#, python-format
msgid "Create or select %(container_type)s"
msgstr "Crear o seliger %(container_type)s"

#: pyanaconda/ui/gui/spokes/custom.py:117
msgid "Device reconfiguration failed. <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:119
msgid ""
"Storage configuration reset due to unrecoverable error. <a href=\"\">Click "
"for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:143
#: pyanaconda/ui/gui/spokes/custom.glade.h:12
msgid "MANUAL PARTITIONING"
msgstr "PARTITIONAMENTO MANUAL"

#: pyanaconda/ui/gui/spokes/custom.py:369
#, python-format
msgctxt "GUI|Custom Partitioning"
msgid "%d _storage device selected"
msgid_plural "%d _storage devices selected"
msgstr[0] "%d di_spositovo de immagazinage seligite"
msgstr[1] "%d di_spositovos de immagazinage seligite"

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/ui/gui/spokes/custom.py:522
#, python-format
msgid ""
"When you create mount points for your %(name)s %(version)s installation, "
"you'll be able to view their details here."
msgstr ""
"Quando tu crea le punctos de montage pro le installation de %(name)s "
"%(version)s, lor detalios se monstrara hic."

#: pyanaconda/ui/gui/spokes/custom.py:676
#, python-format
msgid ""
"/boot/efi must be on a device of type %(oneFsType)s or %(anotherFsType)s"
msgstr ""

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/ui/gui/spokes/custom.py:680
#, python-format
msgid "%(fs)s must be on a device of type %(type)s"
msgstr "%(fs)s debe resider sur un dispositivo del typo %(type)s"

#: pyanaconda/ui/gui/spokes/custom.py:683
#: pyanaconda/ui/gui/spokes/custom.py:685
#, python-format
msgid "%s cannot be encrypted"
msgstr "%s non pote esser cifrate"

#: pyanaconda/ui/gui/spokes/custom.py:687
msgid "You must create a new file system on the root device."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:692
#, python-format
msgid "Device does not support RAID level selection %s."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:800
msgid "Device resize request failed. <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:873
msgid "Device reformat request failed. <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1239
#, python-format
msgid "Specified name %s already in use."
msgstr "Nomine specificate %s jam es in uso."

#: pyanaconda/ui/gui/spokes/custom.py:1397
msgid "No disks assigned"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1403
#, python-format
msgctxt "GUI|Custom Partitioning|Devices"
msgid " and %d other"
msgid_plural " and %d others"
msgstr[0] ""

#: pyanaconda/ui/gui/spokes/custom.py:1464
msgid "The container is encrypted."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1507
msgid ""
"The space available to this mount point can be changed by modifying the "
"volume below."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1509
msgid "This file system may not be resized."
msgstr "Iste systema de files non se pote redimensionar."

#: pyanaconda/ui/gui/spokes/custom.py:1565
msgid ""
"Error checking storage configuration.  <a href=\"\">Click for details</a> or "
"press Done again to continue."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1567
msgid ""
"Warning checking storage configuration.  <a href=\"\">Click for details</a> "
"or press Done again to continue."
msgstr ""

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/ui/gui/spokes/custom.py:1675
#, python-format
msgid "Added new %(type)s to existing container %(name)s."
msgstr "Il ha addite le nove %(type)s al container existente %(name)s."

#: pyanaconda/ui/gui/spokes/custom.py:1686
msgid "Failed to add new device. <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1691
msgid "Invalid partition size set. Use a valid integer."
msgstr ""
"Le dimension del partition es incorrecte. Usa un numero integre valide."

#: pyanaconda/ui/gui/spokes/custom.py:1802
msgid "Device removal request failed. <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1992
msgid "No disks selected. Keeping previous disk set."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2050
msgid "No disks selected. Not saving changes."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2058
#, python-format
msgid "Volume Group name %s is already in use. Not saving changes."
msgstr ""
"Le nomine de gruppo de volumines %s es jam in uso. Le cambiamentos non "
"essera servate."

#: pyanaconda/ui/gui/spokes/custom.py:2090
#, python-format
msgid "(%s free)"
msgstr "(%s libere)"

#: pyanaconda/ui/gui/spokes/custom.py:2247
#, python-format
msgid ""
"This Software RAID array is missing %(missingMembers)d of %(totalMembers)d "
"member partitions. You can remove it or select a different device."
msgstr ""
"%(missingMembers)d de %(totalMembers)d partitiones manca in iste array RAID "
"de software. On pote remover lo o seliger un altere dispositivo."

#: pyanaconda/ui/gui/spokes/custom.py:2253
#, python-format
msgid ""
"This LVM Volume Group is missing %(missingPVs)d of %(totalPVs)d physical "
"volumes. You can remove it or select a different device."
msgstr ""
"%(missingPVs)d de %(totalPVs)d volumines physic manca in iste gruppo de "
"volumines LVM. On pote remover lo o seliger un altere dispositivo."

#: pyanaconda/ui/gui/spokes/custom.py:2335
msgid "No disks selected."
msgstr "Nulle disco seligite."

#: pyanaconda/ui/gui/spokes/custom.py:2340
msgid "Not enough free space on selected disks."
msgstr "Il non ha satis de spatio libere sur le discos seligite."

#: pyanaconda/ui/gui/spokes/custom.py:2345
#: pyanaconda/ui/gui/spokes/custom.py:2351
#: pyanaconda/ui/gui/spokes/custom.py:2376
msgid "Automatic partitioning failed. <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2590
msgid ""
"Continuing with this action will reset all your partitioning selections to "
"their current on-disk state."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2598
msgid "_Reset selections"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2598
msgid "_Preserve current selections"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2677
msgid ""
"Failed to unlock encrypted block device. <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:393
msgctxt "GUI|Spoke"
msgid "DATE & _TIME"
msgstr "DATA E _HORA"

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:530
msgid "Restoring hardware time..."
msgstr "Restablimento de tempore del hardware…"

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:544
msgid "Invalid timezone"
msgstr "Fuso horari incorrecte"

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:715
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:729
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:911
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:27
msgid "PM"
msgstr "PM"

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:717
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:726
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:910
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:913
msgid "AM"
msgstr "AM"

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:1065
msgid "You need to set up networking first if you want to use NTP"
msgstr "Le connexion al rete debe esser configurate ante le uso de NTP"

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:1069
msgid "You have no working NTP server configured"
msgstr "Il non ha configurate nulle servitores NTP operante"

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:1072
msgid "NTP service failed to start."
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:1075
msgid "NTP service failed to stop."
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.py:576
msgctxt "GUI|Spoke"
msgid "_INSTALLATION DESTINATION"
msgstr "DESTINATION DEL _INSTALLATION"

#: pyanaconda/ui/gui/spokes/filter.py:711
#, python-format
msgctxt "GUI|Installation Destination|Filter"
msgid "%d _storage device selected"
msgid_plural "%d _storage devices selected"
msgstr[0] "%d dispositovo de _immagazinage selectionate"
msgstr[1] "%d dispositovos de _immagazinage selectionate"

#: pyanaconda/ui/gui/spokes/keyboard.py:52
#, python-format
msgid "%s to switch layouts."
msgstr "%s pro cambiar le dispositiones."

#: pyanaconda/ui/gui/spokes/keyboard.py:276
msgctxt "GUI|Spoke"
msgid "_KEYBOARD"
msgstr "_CLAVIERO"

#: pyanaconda/ui/gui/spokes/keyboard.py:374
msgid "Testing layouts configuration not available."
msgstr "Le proba de dispositiones de clavieros non es disponibile."

#: pyanaconda/ui/gui/spokes/keyboard.py:382
msgid "Getting list of layouts..."
msgstr "Obtenimento del lista de dispositiones..."

#: pyanaconda/ui/gui/spokes/keyboard.py:435
msgid ""
"Keyboard layouts are not supported when using VNC.\n"
"However the settings will be used after the installation."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.py:445
msgid "Layout switching not configured."
msgstr "Le cambiamento de disposition non es configurate."

#: pyanaconda/ui/gui/spokes/langsupport.py:57
msgctxt "GUI|Spoke"
msgid "_LANGUAGE SUPPORT"
msgstr "SUPPORTO DE _LINGUAS"

#: pyanaconda/ui/gui/spokes/network.py:96
msgid "Status unknown (missing)"
msgstr "Stato incognite (absente)"

#: pyanaconda/ui/gui/spokes/network.py:99
msgid "Status unknown"
msgstr "Stato incognite"

#: pyanaconda/ui/gui/spokes/network.py:101
msgid "Unmanaged"
msgstr "Non gerite"

#: pyanaconda/ui/gui/spokes/network.py:104
#: pyanaconda/ui/gui/spokes/network.py:108
msgid "Unavailable"
msgstr "Nondisponibile"

#: pyanaconda/ui/gui/spokes/network.py:106
msgid "Firmware missing"
msgstr "Firmware mancante"

#: pyanaconda/ui/gui/spokes/network.py:112
msgid "Cable unplugged"
msgstr "Cablo disconnectite"

#: pyanaconda/ui/gui/spokes/network.py:114
msgid "Disconnected"
msgstr "Disconnectite"

#: pyanaconda/ui/gui/spokes/network.py:119
msgid "Connecting"
msgstr "Connectente"

#: pyanaconda/ui/gui/spokes/network.py:121
msgid "Authentication required"
msgstr "Authentication necessari"

#: pyanaconda/ui/gui/spokes/network.py:123
msgid "Connected"
msgstr "Connectite"

#: pyanaconda/ui/gui/spokes/network.py:125
msgid "Disconnecting"
msgstr "Disconnectente"

#: pyanaconda/ui/gui/spokes/network.py:127
msgid "Connection failed"
msgstr "Connexion fallite"

#: pyanaconda/ui/gui/spokes/network.py:327
msgid "Ethernet"
msgstr "Ethernet"

#: pyanaconda/ui/gui/spokes/network.py:328
msgid "Wireless"
msgstr "Sin filo"

#: pyanaconda/ui/gui/spokes/network.py:329
#: pyanaconda/ui/gui/spokes/network.glade.h:1
msgid "Bond"
msgstr "Interfacie aggregate"

#: pyanaconda/ui/gui/spokes/network.py:330
#: pyanaconda/ui/gui/spokes/network.glade.h:4
msgid "VLAN"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:331
#: pyanaconda/ui/gui/spokes/network.glade.h:3
msgid "Team"
msgstr "Equipa"

#: pyanaconda/ui/gui/spokes/network.py:332
#: pyanaconda/ui/gui/spokes/network.glade.h:2
msgid "Bridge"
msgstr ""

#. TRANSLATORS: ethernet cable is unplugged
#: pyanaconda/ui/gui/spokes/network.py:821
msgid "unplugged"
msgstr "disconnectite"

#: pyanaconda/ui/gui/spokes/network.py:917
#: pyanaconda/ui/gui/spokes/network.glade.h:16
msgid "IPv4 Address"
msgstr "Adresse IPv4"

#: pyanaconda/ui/gui/spokes/network.py:918
#: pyanaconda/ui/gui/spokes/network.glade.h:15
msgid "IPv6 Address"
msgstr "Adresse IPv6"

#: pyanaconda/ui/gui/spokes/network.py:920
#: pyanaconda/ui/gui/spokes/network.py:922
#: pyanaconda/ui/gui/spokes/network.glade.h:31
msgid "IP Address"
msgstr "Adresse IP"

#: pyanaconda/ui/gui/spokes/network.py:1012
#, python-format
msgid "%d Mb/s"
msgstr "%d Mb/s"

#: pyanaconda/ui/gui/spokes/network.py:1232
msgid "WEP"
msgstr "WEP"

#: pyanaconda/ui/gui/spokes/network.py:1235
msgid "WPA"
msgstr "WPA"

#: pyanaconda/ui/gui/spokes/network.py:1238
msgid "WPA2"
msgstr "WPA2"

#: pyanaconda/ui/gui/spokes/network.py:1242
msgid "Enterprise"
msgstr "Interprisa"

#: pyanaconda/ui/gui/spokes/network.py:1247 widgets/src/SpokeSelector.c:82
#: pyanaconda/ui/gui/spokes/filter.glade.h:3
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:21
msgid "None"
msgstr "Nulle"

#: pyanaconda/ui/gui/spokes/network.py:1388
#: pyanaconda/ui/gui/spokes/network.glade.h:61
msgid "Authentication required by wireless network"
msgstr "Authentication es requirite per rete sin filo"

#: pyanaconda/ui/gui/spokes/network.py:1389
#, python-format
msgid ""
"Passwords or encryption keys are required to access\n"
"the wireless network '%(network_id)s'."
msgstr ""
"Contrasignos o claves cryptographic es necessari \n"
"pro connecter se al rete sin filo '%(network_id)s'."

#: pyanaconda/ui/gui/spokes/network.py:1403
msgctxt "GUI|Network|Secrets Dialog"
msgid "_Password:"
msgstr "Contra_signo:"

#: pyanaconda/ui/gui/spokes/network.py:1411
msgctxt "GUI|Network|Secrets Dialog"
msgid "_Key:"
msgstr "_Clave:"

#: pyanaconda/ui/gui/spokes/network.py:1421
msgid "User name: "
msgstr "Nomine de usator: "

#: pyanaconda/ui/gui/spokes/network.py:1432
msgid "Identity: "
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1437
msgid "Private key password: "
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1493
msgctxt "GUI|Spoke"
msgid "_NETWORK & HOST NAME"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:54
msgctxt "GUI|Spoke"
msgid "_ROOT PASSWORD"
msgstr "CONTRASIGNO DE SUPE_RUSATOR"

#: pyanaconda/ui/gui/spokes/password.py:82
#: pyanaconda/ui/gui/spokes/password.py:83
msgid "The password is set."
msgstr "Le contrasigno es definite."

#: pyanaconda/ui/gui/spokes/password.py:117
msgid "Root password is set"
msgstr "Le contrasigno de superusator es definite"

#: pyanaconda/ui/gui/spokes/password.py:119
msgid "Root account is disabled"
msgstr "Le conto de superusator es disactivate"

#: pyanaconda/ui/gui/spokes/password.py:121
msgid "Root password is not set"
msgstr "Le contrasigno de superusator non es definite"

#: pyanaconda/ui/gui/spokes/software.py:56
msgctxt "GUI|Spoke"
msgid "_SOFTWARE SELECTION"
msgstr "SELECTION DE _SOFTWARE"

#: pyanaconda/ui/gui/spokes/software.py:106
#: pyanaconda/ui/gui/spokes/source.py:60
msgid "Downloading package metadata..."
msgstr "Discargamento de metadatos de pacchettos..."

#: pyanaconda/ui/gui/spokes/software.py:109
msgid "Downloading group metadata..."
msgstr "Discargamento de metadatos de gruppos..."

#: pyanaconda/ui/gui/spokes/software.py:201
#: pyanaconda/ui/gui/spokes/source.py:561
msgid "Checking software dependencies..."
msgstr "Controlo de dependentias de software..."

#: pyanaconda/ui/gui/spokes/software.py:206
msgid "Error checking software dependencies"
msgstr "Error al controlo de dependentias de software"

#: pyanaconda/ui/gui/spokes/software.py:291
msgid "Invalid environment specified in kickstart"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:300
#: pyanaconda/ui/gui/spokes/software.py:307
msgid "Selected environment is not valid"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:461
msgid ""
"Error checking software dependencies.  <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:571
msgid ""
"The software marked for installation has the following errors.  This is "
"likely caused by an error with your installation source.  You can quit the "
"installer, change your software source, or change your software selections."
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:576
msgctxt "GUI|Software Selection|Error Dialog"
msgid "_Quit"
msgstr "_Quitar"

#: pyanaconda/ui/gui/spokes/software.py:577
msgctxt "GUI|Software Selection|Error Dialog"
msgid "_Modify Software Source"
msgstr "_Modificar le fonte de programmas"

#: pyanaconda/ui/gui/spokes/software.py:578
msgctxt "GUI|Software Selection|Error Dialog"
msgid "Modify _Selections"
msgstr "Modificar _selectiones"

#: pyanaconda/ui/gui/spokes/source.py:59
msgid "Setting up installation source..."
msgstr "Configuration del fonte de installation..."

#: pyanaconda/ui/gui/spokes/source.py:91
msgid "Invalid proxy URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:96
#, python-format
msgid "Invalid proxy protocol: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:102
msgid "Extra characters in proxy URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:106
msgid "Proxy authentication data duplicated"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:244
msgid "This media is good to install from."
msgstr "Iste medio es bon pro installar de illac."

#: pyanaconda/ui/gui/spokes/source.py:246
msgid "This media is not good to install from."
msgstr "Iste medio non es bon pro installar de illac."

#: pyanaconda/ui/gui/spokes/source.py:367
msgctxt "GUI|Spoke"
msgid "_INSTALLATION SOURCE"
msgstr "FONTE DE _INSTALLATION"

#: pyanaconda/ui/gui/spokes/source.py:490
#: pyanaconda/ui/gui/spokes/source.py:716
msgid "Failed to set up installation source; check the repo url"
msgstr ""
"Impossibile de configurar le fonte de installation; verifica le URL de "
"repositorio"

#: pyanaconda/ui/gui/spokes/source.py:565
msgid "Error setting up base repository"
msgstr "Error al configuration del repositorio fundamental"

#: pyanaconda/ui/gui/spokes/source.py:578
msgid "Error setting up ISO file"
msgstr "Error al configuration del file ISO"

#: pyanaconda/ui/gui/spokes/source.py:700
msgid "Testing availability..."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:718
msgid ""
"Failed to set up installation source; check the repo url and proxy settings"
msgstr ""
"Impossibile de configurar le fonte de installation; verifica le URL de "
"repositorio e parametros de proxy"

#: pyanaconda/ui/gui/spokes/source.py:745
#, python-format
msgid "Device: %s"
msgstr "Dispositivo: %s"

#: pyanaconda/ui/gui/spokes/source.py:746
#, python-format
msgid "Label: %s"
msgstr "Etiquetta: %s"

#: pyanaconda/ui/gui/spokes/source.py:921
msgid "You need to configure the network to use a network installation source."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:984
msgid "URL is empty"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:988
msgid "Invalid URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:1005
msgid "Protocol in URL does not match selected protocol"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:1008
msgid "NFS server is empty"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:1015
msgid "Invalid host name"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:1018
msgid "Remote directory is required"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:1039
msgid "Duplicate repository names."
msgstr "Duplicitate de nomines de repositorios."

#: pyanaconda/ui/gui/spokes/source.py:1046
msgid "Empty repository name"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:1049
msgid "Invalid repository name"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:1055
msgid "Repository name conflicts with internal repository name."
msgstr ""
"Le nomine de repositorio conflige con le nomine de un repositorio interne."

#: pyanaconda/ui/gui/spokes/storage.py:121
msgid "Please wait... software metadata still loading."
msgstr ""
"Attende... le cargamento de metadatos de software es ancora in processo."

#: pyanaconda/ui/gui/spokes/storage.py:122
#, python-format
msgid ""
"Your current <a href=\"\" title=\"%(tooltip)s\"><b>%(product)s</b> software "
"selection</a> requires <b>%(total)s</b> of available space, including <b>"
"%(software)s</b> for software and <b>%(swap)s</b> for swap space."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:172
#, python-format
msgid "%s The disks you've selected have the following amounts of free space:"
msgstr ""
"%s In le discos que ha seligite es le sequente amontas de spatio libere:"

#: pyanaconda/ui/gui/spokes/storage.py:180
#, python-format
msgid ""
"<b>You don't have enough space available to install %s</b>.  You can shrink "
"or remove existing partitions via our guided reclaim space tool, or you can "
"adjust your partitions on your own in the custom partitioning interface."
msgstr ""
"<b>Le spatio pro le installation de %s es insufficiente</b>. On pote reducer "
"o eliminar le partitiones existente via le guida pro recuperar spatio, o on "
"pote adjustar le partitiones via le interfacie del partitionamento "
"personalisate."

#: pyanaconda/ui/gui/spokes/storage.py:204
#, python-format
msgid ""
"  You don't have enough space available to install <b>%(product)s</b>, even "
"if you used all of the free space available on the selected disks."
msgstr ""

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/ui/gui/spokes/storage.py:214
#, python-format
msgid ""
"<b>You don't have enough space available to install %(productName)s</b>, "
"even if you used all of the free space available on the selected disks.  You "
"could add more disks for additional space, modify your software selection to "
"install a smaller version of <b>%(productName)s</b>, or quit the installer."
msgstr ""
"<b>Le spatio pro le installation de %(productName)s es insufficiente</b>, "
"etiam si tu usate tote le spatio disponibile sur le discos seligite. On pote "
"connecter altere discos pro obtener spatio additional, modificar le "
"selection de programma pro installar un variante plus parve de "
"<b>%(productName)s</b>, o quitar le installator."

#: pyanaconda/ui/gui/spokes/storage.py:235
msgctxt "GUI|Spoke"
msgid "INSTALLATION _DESTINATION"
msgstr "_DESTINATION DEL INSTALLATION"

#: pyanaconda/ui/gui/spokes/storage.py:359
msgid "Saving storage configuration..."
msgstr "Salveguarda del configuration de immagazinage..."

#: pyanaconda/ui/gui/spokes/storage.py:363
msgid "Passphrase for autopart encryption not specified."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:372
#: pyanaconda/ui/gui/spokes/storage.py:384
msgid "Failed to save storage configuration..."
msgstr "Impossibile de salveguardar le configuration de immagazinage..."

#: pyanaconda/ui/gui/spokes/storage.py:444
#: pyanaconda/ui/gui/spokes/storage.py:751
#: pyanaconda/ui/gui/spokes/storage.py:770
msgid "Formatting DASDs"
msgstr "Formatation de DASD"

#: pyanaconda/ui/gui/spokes/storage.py:590
msgid ""
"Error checking storage configuration.  <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:593
msgid ""
"Warning checking storage configuration.  <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:637
#, python-format
msgid ""
"FCP device %(hba_id)s\n"
"WWPN %(wwpn)s\n"
"LUN %(lun)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:640
#, python-format
msgid "NVDIMM device %(namespace)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:649
#, python-format
msgid "%s free"
msgstr "%s libere"

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/ui/gui/spokes/storage.py:699
#, python-format
msgid "%(count)d disk selected; %(capacity)s capacity; %(free)s free"
msgid_plural "%(count)d disks selected; %(capacity)s capacity; %(free)s free"
msgstr[0] "%(count)d disco seligite; %(capacity)s capacitate; %(free)s libere"
msgstr[1] "%(count)d discos seligite; %(capacity)s capacitate; %(free)s libere"

#: pyanaconda/ui/gui/spokes/storage.py:812
msgid ""
"You have chosen to skip boot loader installation.  Your system may not be "
"bootable."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:914
msgid "There was a problem with your disk selection. Click here for details."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:1089
msgid "You'll be able to make space available during custom partitioning."
msgstr ""
"Le spatio pote esser facite disponibile durante le partitionamento "
"personalisate."

#: pyanaconda/ui/gui/spokes/storage.py:1109
msgid ""
"The following errors were encountered when checking your disk selection. You "
"can modify your selection or quit the installer."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:1114
#: pyanaconda/ui/gui/spokes/storage.py:1135
msgctxt "GUI|Storage|Error Dialog"
msgid "_Quit"
msgstr "_Quitar"

#: pyanaconda/ui/gui/spokes/storage.py:1115
msgctxt "GUI|Storage|Error Dialog"
msgid "_Modify Disk Selection"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:1130
msgid ""
"The following errors were encountered when checking your storage "
"configuration.  You can modify your storage layout or quit the installer."
msgstr ""
"Le sequente errores se ha incontrate durante le controlo del configuration "
"de immagazinage. Il pote modificar le schema de immagazinage o quitar le "
"installator."

#: pyanaconda/ui/gui/spokes/storage.py:1136
msgctxt "GUI|Storage|Error Dialog"
msgid "_Modify Storage Layout"
msgstr "_Modificar le schema de immagazinage"

#: pyanaconda/ui/gui/spokes/storage.py:1150
msgid ""
"The following warnings were encountered when checking your storage "
"configuration.  These are not fatal, but you may wish to make changes to "
"your storage layout."
msgstr ""
"Le sequente avisos se ha incontrate durante le controlo del configuration de "
"immagazinage. Istos non es fatal, ma tu pote vol modificar le schema de "
"immagazinage."

#: pyanaconda/ui/gui/spokes/storage.py:1154
msgid "_OK"
msgstr "_OK"

#: pyanaconda/ui/gui/spokes/user.py:70
#, python-format
msgid "Invalid group name: %s"
msgstr "Le nomine de gruppo invalide: %s"

#: pyanaconda/ui/gui/spokes/user.py:213
msgctxt "GUI|Spoke"
msgid "_USER CREATION"
msgstr "CREATION DE _USATOR"

#: pyanaconda/ui/gui/spokes/user.py:299 pyanaconda/ui/gui/spokes/user.py:300
#: pyanaconda/ui/gui/spokes/user.py:336 pyanaconda/ui/gui/spokes/user.py:337
msgid "The password was set by kickstart."
msgstr "Le contrasigno esseva configurate per kickstart."

#: pyanaconda/ui/gui/spokes/user.py:328
msgid "Full name cannot contain colon characters"
msgstr "Le nomine integre non pote continer characteres de duo punctos"

#: pyanaconda/ui/gui/spokes/user.py:548
msgid "Invalid user name"
msgstr ""

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/ui/gui/spokes/welcome.py:224
#, python-format
msgid "WELCOME TO %(name)s %(version)s."
msgstr "BENVENITE A %(name)s %(version)s."

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.py:275
#, python-format
msgid ""
"The following nodes have been discovered using the iSCSI initiator <b>"
"%(initiatorName)s</b> using the target IP address <b>%(targetAddress)s</b>.  "
"Please select which nodes you wish to log into:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.py:85
msgctxt "GUI|Advanced Storage|NVDIM"
msgid "No device to be reconfigured selected."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.py:160
msgid "Rescanning disks finished."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:171
msgid "DATA"
msgstr "DATOS"

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/ui/gui/spokes/lib/accordion.py:293
#, python-format
msgid ""
"You haven't created any mount points for your %(product)s %(version)s "
"installation yet.  You can:"
msgstr ""
"Il non ha ancora create nulle punctos de montage pro le installation de "
"%(product)s %(version)s. On pote:"

#: pyanaconda/ui/gui/spokes/lib/accordion.py:302
msgid "_Click here to create them automatically."
msgstr "_Clicca ci pro crear los automaticamente."

#: pyanaconda/ui/gui/spokes/lib/accordion.py:327
msgid "Create new mount points by clicking the '+' button."
msgstr "Crear nove punctos de montage per cliccar sur le button '+'."

#: pyanaconda/ui/gui/spokes/lib/accordion.py:335
msgid ""
"Or, assign new mount points to existing partitions after selecting them "
"below."
msgstr ""
"O, assignar nove punctos de montage al partitiones existente post lor "
"selection in basso."

#: pyanaconda/ui/gui/spokes/lib/accordion.py:340
msgid "_New mount points will use the following partitioning scheme:"
msgstr "_Nove punctos de montage utilisara le sequente schema de partitiones:"

#: pyanaconda/ui/gui/spokes/lib/cart.py:134
#, python-format
msgid ""
"<b>%(count)d disk; %(size)s capacity; %(free)s free space</b> (unpartitioned "
"and in file systems)"
msgid_plural ""
"<b>%(count)d disks; %(size)s capacity; %(free)s free space</b> "
"(unpartitioned and in file systems)"
msgstr[0] ""

#: pyanaconda/ui/gui/spokes/lib/cart.py:193
msgctxt "GUI|Selected Disks Dialog"
msgid "_Do not install boot loader"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.py:196
#: pyanaconda/ui/gui/spokes/lib/cart.glade.h:8
msgctxt "GUI|Selected Disks Dialog"
msgid "_Set as Boot Device"
msgstr "_Definir como dispositivo de initio"

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:62
#, python-format
msgid ""
"The RAID level you have selected (%(level)s) requires more disks (%(min)d) "
"than you currently have selected (%(count)d)."
msgstr ""
"Le nivello RAID seligite (%(level)s) require plus de discos (%(min)d) que "
"ora es seligite (%(count)d)."

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:66
#, python-format
msgid "CONFIGURE %(container_type)s"
msgstr "CONFIGURAR %(container_type)s"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:67
#, python-format
msgid ""
"Please create a name for this %(container_type)s and select at least one "
"disk below."
msgstr ""
"Crea un nomine pro iste %(container_type)s e seligie al minus un disco in "
"basso."

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:70
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:71
msgid "Volume Group"
msgstr "Gruppo de volumines"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:72
msgid "Volume"
msgstr "Volumine"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:112
msgid "Cannot relabel already existing file system."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:117
msgid "Cannot set label on file system."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:119
msgid "Unacceptable label format for file system."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:129
msgid "That mount point is already in use. Try something else?"
msgstr "Iste puncto de montage ja es in uso. Probar qualcosa altere?"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:131
msgid "Please enter a valid mount point."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:133
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:145
msgid "That mount point is invalid. Try something else?"
msgstr "Iste puncto de montage non es valide. Probar qualcosa altere?"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:286
msgid "container"
msgstr "container"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:358
#, python-format
msgctxt "GUI|Custom Partitioning|Confirm Delete Dialog"
msgid "Delete _all file systems which are only used by %s."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:368
#, python-format
msgid "Are you sure you want to delete all of the data on %s?"
msgstr "Eliminar tote le datos sur %s?"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:370
#, python-format
msgid ""
"Are you sure you want to delete all of the data on %s, including snapshots "
"and/or subvolumes?"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:647
msgid "Invalid container name"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/detailederror.py:49
msgctxt "GUI|Detailed Error Dialog"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/lib/detailederror.py:59
msgctxt "GUI|Detailed Error Dialog"
msgid "_Quit"
msgstr "_Quitar"

#: pyanaconda/ui/gui/spokes/lib/resize.py:58
msgid "Preserve"
msgstr "Preservar"

#: pyanaconda/ui/gui/spokes/lib/resize.py:59
msgid "Shrink"
msgstr "Reducer"

#: pyanaconda/ui/gui/spokes/lib/resize.py:60
msgid "Delete"
msgstr "Deler"

#: pyanaconda/ui/gui/spokes/lib/resize.py:87
#, python-format
msgid "Installation requires a total of <b>%s</b> for system data."
msgstr "Le installation require un total de <b>%s</b> pro le datos de systema."

#: pyanaconda/ui/gui/spokes/lib/resize.py:119
msgid "This device contains the installation source."
msgstr "Iste dispositivo contine le fonte de installation."

#: pyanaconda/ui/gui/spokes/lib/resize.py:166
#, python-format
msgid "%(freeSize)s of %(devSize)s"
msgstr "%(freeSize)s de %(devSize)s"

#: pyanaconda/ui/gui/spokes/lib/resize.py:173
msgid "Not resizeable"
msgstr "Non redimensionabile"

#: pyanaconda/ui/gui/spokes/lib/resize.py:197
msgid "Free space"
msgstr "Liberar spatio"

#: pyanaconda/ui/gui/spokes/lib/resize.py:219
msgid ""
"You can remove existing file systems you no longer need to free up space for "
"this installation.  Removing a file system will permanently delete all of "
"the data it contains."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:225
msgid ""
"There is also free space available in pre-existing file systems.  While it's "
"risky and we recommend you back up your data first, you can recover that "
"free disk space and make it available for this installation below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:235
#, python-format
msgid "<b>%(count)s disk; %(size)s reclaimable space</b> (in file systems)"
msgid_plural ""
"<b>%(count)s disks; %(size)s reclaimable space</b> (in file systems)"
msgstr[0] ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:242
#, python-format
msgid "Total selected space to reclaim: <b>%s</b>"
msgstr "Spatio total seligite a recuperar: <b>%s</b>"

#: pyanaconda/ui/gui/spokes/lib/resize.py:457
#: pyanaconda/ui/gui/spokes/lib/resize.py:462
#: pyanaconda/ui/gui/spokes/lib/resize.glade.h:13
msgctxt "GUI|Reclaim Dialog"
msgid "Delete _all"
msgstr "Deler _toto"

#: pyanaconda/ui/gui/spokes/lib/resize.py:459
msgctxt "GUI|Reclaim Dialog"
msgid "Preserve _all"
msgstr "Preservar _toto"

#: widgets/src/BaseWindow.c:153
msgid "DISTRIBUTION INSTALLATION"
msgstr "INSTALLATION DEL DISTRIBUTION"

#: widgets/src/BaseWindow.c:154
msgid "SPOKE NAME"
msgstr "NOMINE DEL SUCCURSAL"

#: widgets/src/BaseWindow.c:155
msgid "PRE-RELEASE / TESTING"
msgstr "PRE-EDITION / PROBATION"

#: widgets/src/DiskOverview.c:101
msgid "New Device"
msgstr "Nove dispositivo"

#: widgets/src/DiskOverview.c:103 widgets/src/DiskOverview.c:104
msgid "0 MB"
msgstr "0 MB"

#: widgets/src/MountpointSelector.c:84
msgid "0 GB"
msgstr "0 GB"

#: widgets/src/SpokeSelector.c:83
msgid "New Selector"
msgstr "Nove selector"

#: widgets/src/SpokeWindow.c:57
msgid "_Done"
msgstr "Pr_este"

#: widgets/src/StandaloneWindow.c:50
msgid "_Continue"
msgstr "_Continuar"

#: widgets/src/LayoutIndicator.c:33
#, c-format
msgid "Current layout: '%s'. Click to switch to the next layout."
msgstr ""
"Le disposition actual: '%s'. Clicca pro cambiar lo al disposition sequente."

#: widgets/src/LayoutIndicator.c:34
#, c-format
msgid "Current layout: '%s'. Add more layouts to enable switching."
msgstr ""
"Le disposition actual: '%s'. Adde dispositiones pro permitter cambiamentos."

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#: pyanaconda/ui/gui/spokes/advanced_user.glade.h:1
msgctxt "GUI|Advanced User"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/advanced_user.glade.h:2
msgctxt "GUI|Advanced User"
msgid "_Save Changes"
msgstr "_Salveguardar modificationes"

#: pyanaconda/ui/gui/spokes/advanced_user.glade.h:3
msgid "ADVANCED USER CONFIGURATION"
msgstr "CONFIGURATION AVANTIATE DE USATORES"

#: pyanaconda/ui/gui/spokes/advanced_user.glade.h:4
msgctxt "GUI|Advanced User"
msgid "Home _directory:"
msgstr "_Directorio personal:"

#: pyanaconda/ui/gui/spokes/advanced_user.glade.h:5
msgid "User and Group IDs"
msgstr "Identificatores de usator e gruppo"

#: pyanaconda/ui/gui/spokes/advanced_user.glade.h:6
msgctxt "GUI|Advanced User"
msgid "Specify a _user ID manually:"
msgstr "Specificar manualmente un identificator de _usator:"

#: pyanaconda/ui/gui/spokes/advanced_user.glade.h:7
msgctxt "GUI|Advanced User"
msgid "Specify a _group ID manually:"
msgstr "Specificar manualmente un identificator de _gruppo:"

#: pyanaconda/ui/gui/spokes/advanced_user.glade.h:8
msgid "Group Membership"
msgstr "Adhesion de gruppo"

#: pyanaconda/ui/gui/spokes/advanced_user.glade.h:9
msgctxt "GUI|Advanced User"
msgid "_Add user to the following groups:"
msgstr "_Adder le usator al sequente gruppos:"

#: pyanaconda/ui/gui/spokes/advanced_user.glade.h:10
msgid "Example:"
msgstr "Exemplo:"

#: pyanaconda/ui/gui/spokes/advanced_user.glade.h:11
msgid "wheel, my-team (1245), project-x (29935)"
msgstr "wheel, my-team (1245), project-x (29935)"

#: pyanaconda/ui/gui/spokes/advanced_user.glade.h:12
msgid "Tip:"
msgstr "Aviso:"

#: pyanaconda/ui/gui/spokes/advanced_user.glade.h:13
msgid ""
"You may input a comma-separated list of group names and group IDs here. "
"Groups that do not already exist will be created; specify their GID in "
"parentheses. "
msgstr ""

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:1
msgctxt "GUI|Date and Time|NTP"
msgid "_Cancel"
msgstr "_Cancellar"

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:2
msgctxt "GUI|Date and Time|NTP"
msgid "_OK"
msgstr "_OK"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:3
msgid "Add and mark for usage NTP servers"
msgstr "Adder servitores NTP e marcar los pro le uso"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:4
msgid "New NTP Server"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:5
msgid "Add NTP Server"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:7
msgid "Working"
msgstr "Operational"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:8
msgid "Use"
msgstr "Usar"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:9
msgid "Configure NTP"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:10
msgid "DATE & TIME"
msgstr "DATA E HORA"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:11
msgctxt "GUI|Date and Time"
msgid "_Region:"
msgstr "_Region:"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:12
msgid "Region"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:13
msgctxt "GUI|Date and Time"
msgid "_City:"
msgstr "_Citate:"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:14
msgid "City"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:15
msgctxt "GUI|Date and Time"
msgid "_Network Time"
msgstr "Hora de re_te"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:16
msgid "Use Network Time"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:17
msgid "Hours"
msgstr ""

#. TRANSLATORS: This is the separator between hours and minutes, like in HH:MM
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:19
msgid ":"
msgstr ":"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:20
msgid "Minutes"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:21
msgid "Hour Up"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:22
msgid "Hour Down"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:23
msgid "AM/PM Up"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:24
msgid "Minutes Up"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:25
msgid "AM/PM Down"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:26
msgid "Minutes Down"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:28
msgctxt "GUI|Date and Time"
msgid "24-_hour"
msgstr "24 _horas"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:29
msgctxt "GUI|Date and Time"
msgid "_AM/PM"
msgstr "_AM/PM"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade.h:30
msgid "Set Date & Time"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:1
#: pyanaconda/ui/gui/spokes/storage.glade.h:14
msgid "INSTALLATION DESTINATION"
msgstr "DESTINATION DEL INSTALLATION"

#: pyanaconda/ui/gui/spokes/filter.glade.h:2
msgctxt "GUI|Installation Destination|Filter|Search"
msgid "Search _By:"
msgstr "_Cercar per:"

#: pyanaconda/ui/gui/spokes/filter.glade.h:4
msgid "Port / Target / LUN #"
msgstr "Porto / Target / LUN #"

#: pyanaconda/ui/gui/spokes/filter.glade.h:5
msgid "Target WWID"
msgstr "WWID de scopo"

#: pyanaconda/ui/gui/spokes/filter.glade.h:6
msgctxt "GUI|Installation Destination|Filter|Search|Port Target LUN"
msgid "_Port:"
msgstr "_Porto:"

#: pyanaconda/ui/gui/spokes/filter.glade.h:7
msgctxt "GUI|Installation Destination|Filter|Search|Port Target LUN"
msgid "_Target:"
msgstr "_Scopo:"

#: pyanaconda/ui/gui/spokes/filter.glade.h:8
msgctxt "GUI|Installation Destination|Filter|Search|Port Target LUN"
msgid "_LUN:"
msgstr "_LUN:"

#: pyanaconda/ui/gui/spokes/filter.glade.h:9
msgctxt "GUI|Installation Destination|Filter|Search|WWID"
msgid "_WWID:"
msgstr "_WWID:"

#: pyanaconda/ui/gui/spokes/filter.glade.h:10
msgctxt "GUI|Installation Destination|Filter|Search"
msgid "Search Res_ults:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:11
#: pyanaconda/ui/gui/spokes/keyboard.glade.h:18
#: pyanaconda/ui/gui/spokes/source.glade.h:34
#: pyanaconda/ui/gui/spokes/lib/cart.glade.h:5
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:8
#: pyanaconda/ui/gui/spokes/lib/resize.glade.h:6
msgid "Name"
msgstr "Nomine"

#: pyanaconda/ui/gui/spokes/filter.glade.h:12
msgid "WWID"
msgstr "WWID"

#: pyanaconda/ui/gui/spokes/filter.glade.h:13
#: pyanaconda/ui/gui/spokes/lib/cart.glade.h:6
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:9
msgid "Capacity"
msgstr "Capacitate"

#: pyanaconda/ui/gui/spokes/filter.glade.h:14
msgid "Interconnect"
msgstr "Interconnexion"

#: pyanaconda/ui/gui/spokes/filter.glade.h:15
msgid "Model"
msgstr "Modello"

#: pyanaconda/ui/gui/spokes/filter.glade.h:16
msgid "LUN"
msgstr "LUN"

#: pyanaconda/ui/gui/spokes/filter.glade.h:17
msgid "Port"
msgstr "Porto"

#: pyanaconda/ui/gui/spokes/filter.glade.h:18
msgid "Target"
msgstr "Scopo"

#: pyanaconda/ui/gui/spokes/filter.glade.h:19
msgid "Vendor"
msgstr "Venditor"

#: pyanaconda/ui/gui/spokes/filter.glade.h:20
msgid "Namespace"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:21
msgid "Mode"
msgstr "Modo"

#: pyanaconda/ui/gui/spokes/filter.glade.h:22
msgctxt "GUI|Installation Destination|Filter"
msgid "Searc_h"
msgstr "_Cercar"

#: pyanaconda/ui/gui/spokes/filter.glade.h:23
msgctxt "GUI|Installation Destination|Filter|Multipath"
msgid "Filter _By:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:24
msgctxt "GUI|Installation Destination|Filter|Multipath|Vendor"
msgid "Show Only _Devices From:"
msgstr "_Monstrar solmente dispositivos de:"

#: pyanaconda/ui/gui/spokes/filter.glade.h:25
msgctxt "GUI|Installation Destination|Filter|Multipath|Interconnect"
msgid "Show Only _Devices With:"
msgstr "_Monstrar solmente dispositivos con:"

#: pyanaconda/ui/gui/spokes/filter.glade.h:26
msgctxt "GUI|Installation Destination|Filter|Multipath|WWID"
msgid "Show Only _Devices Containing:"
msgstr "_Monstrar solmente dispositivos qual contine:"

#: pyanaconda/ui/gui/spokes/filter.glade.h:27
msgid "Paths"
msgstr "Percursos"

#: pyanaconda/ui/gui/spokes/filter.glade.h:28
msgctxt "GUI|Installation Destination|Filter"
msgid "_Multipath Devices"
msgstr "Dispositivos _multivie"

#: pyanaconda/ui/gui/spokes/filter.glade.h:29
msgctxt "GUI|Installation Destination|Filter|Other"
msgid "Filter _By:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:30
msgid "ID"
msgstr "ID"

#: pyanaconda/ui/gui/spokes/filter.glade.h:31
msgctxt "GUI|Installation Destination|Filter|Other|Vendor"
msgid "Show Only _Devices From:"
msgstr "_Monstrar solmente dispositivos de:"

#: pyanaconda/ui/gui/spokes/filter.glade.h:32
msgctxt "GUI|Installation Destination|Filter|Other|Interconnect"
msgid "Show Only _Devices With:"
msgstr "_Monstrar solmente dispositivos con:"

#: pyanaconda/ui/gui/spokes/filter.glade.h:33
msgctxt "GUI|Installation Destination|Filter|Other|ID"
msgid "Show Only _Devices Containing:"
msgstr "_Monstrar solmente dispositivos qual contine:"

#: pyanaconda/ui/gui/spokes/filter.glade.h:34
msgid "Identifier"
msgstr "Identificator"

#: pyanaconda/ui/gui/spokes/filter.glade.h:35
msgctxt "GUI|Installation Destination|Filter"
msgid "_Other SAN Devices"
msgstr "_Altere dispositivos SAN"

#: pyanaconda/ui/gui/spokes/filter.glade.h:36
msgctxt "GUI|Installation Destination|Filter|NVDIMM Devices"
msgid "Filter B_y:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:37
msgctxt "GUI|Installation Destination|Filter|NVDIMM Devices|Mode"
msgid "Show Only Devices in Mo_de:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:38
msgid "Sector"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:39
msgid "Memory"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:40
msgctxt "GUI|Installation Destination|Filter|NVDIMM Devices|Namespace"
msgid "Names_pace:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:41
msgctxt "GUI|Installation Destination|Filter"
msgid "_NVDIMM Devices"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:42
msgctxt "GUI|Installation Destination|Filter|zSeries"
msgid "Filter B_y:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:43
msgid "CCW"
msgstr "CCW"

#: pyanaconda/ui/gui/spokes/filter.glade.h:44
msgid "WWPN"
msgstr "WWPN"

# translation auto-copied from project anaconda, version rhel7-branch,
# document anaconda
#: pyanaconda/ui/gui/spokes/filter.glade.h:45
msgctxt "GUI|Installation Destination|Filter|zSeries|CCW"
msgid "_CCW:"
msgstr "_CCW:"

# translation auto-copied from project anaconda, version rhel7-branch,
# document anaconda
#: pyanaconda/ui/gui/spokes/filter.glade.h:46
msgctxt "GUI|Installation Destination|Filter|zSeries|WWPN"
msgid "_WWPN:"
msgstr "_WWPN:"

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#: pyanaconda/ui/gui/spokes/filter.glade.h:47
msgctxt "GUI|Installation Destination|Filter|zSeries|LUN"
msgid "_LUN:"
msgstr "_LUN:"

#: pyanaconda/ui/gui/spokes/filter.glade.h:48
#: pyanaconda/ui/gui/spokes/lib/summary.glade.h:7
msgid "Type"
msgstr "Typo"

#: pyanaconda/ui/gui/spokes/filter.glade.h:49
msgctxt "GUI|Installation Destination|Filter"
msgid "_zSeries Devices"
msgstr "Dispositivos _zSeries"

#: pyanaconda/ui/gui/spokes/filter.glade.h:50
msgctxt "GUI|Installation Destination|Filter"
msgid "_Add zFCP LUN..."
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:51
msgctxt "GUI|Installation Destination|Filter"
msgid "Add EC_KD DASD..."
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:52
msgctxt "GUI|Installation Destination|Filter"
msgid "Add _iSCSI Target..."
msgstr "Adder scopo _iSCSI…"

#: pyanaconda/ui/gui/spokes/filter.glade.h:53
msgctxt "GUI|Installation Destination|Filter"
msgid "Add FCo_E SAN..."
msgstr "Adder SAN FCo_E..."

#: pyanaconda/ui/gui/spokes/filter.glade.h:54
msgctxt "GUI|Installation Destination|Filter"
msgid "Recon_figure NVDIMM..."
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade.h:55
msgctxt "GUI|Installation Destination|Filter"
msgid "_Refresh List"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:5
msgid "Add device"
msgstr "Adder un dispositivo"

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#: pyanaconda/ui/gui/spokes/network.glade.h:6
msgctxt "GUI|Network|Add Device Dialog"
msgid "_Cancel"
msgstr "_Cancellar"

# translation auto-copied from project anaconda, version rhel6-branch,
# document anaconda
#: pyanaconda/ui/gui/spokes/network.glade.h:7
msgctxt "GUI|Network|Add Device Dialog"
msgid "_Add"
msgstr "_Adder"

#: pyanaconda/ui/gui/spokes/network.glade.h:8
msgctxt "GUI|Network|Add Device Dialog"
msgid "_Select the type of device you wish to add"
msgstr "_Seliger le typo de dispositivo pro adder"

#: pyanaconda/ui/gui/spokes/network.glade.h:9
msgid "NETWORK & HOST NAME"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:10
msgid ""
"Please use the live desktop environment's tools for customizing your network "
"configuration.  You can set the host name here."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:11
msgid "Slaves"
msgstr "Sclavo"

#: pyanaconda/ui/gui/spokes/network.glade.h:12
msgid "DNS"
msgstr "DNS"

#: pyanaconda/ui/gui/spokes/network.glade.h:13
msgid "Default Route"
msgstr "Route predifinite"

#: pyanaconda/ui/gui/spokes/network.glade.h:14
msgid "Subnet Mask"
msgstr "Masca de subrete"

#: pyanaconda/ui/gui/spokes/network.glade.h:17
msgid "Speed"
msgstr "Velocitate"

#: pyanaconda/ui/gui/spokes/network.glade.h:18
msgid "Hardware Address"
msgstr "Adresse de apparato"

#: pyanaconda/ui/gui/spokes/network.glade.h:19
msgid "Parent"
msgstr "Parente"

#: pyanaconda/ui/gui/spokes/network.glade.h:20
msgid "VLAN ID"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:21
msgctxt "GUI|Network|Wired"
msgid "C_onfigure..."
msgstr "C_onfigurar..."

#: pyanaconda/ui/gui/spokes/network.glade.h:22
msgid "Security"
msgstr "Securitate"

#: pyanaconda/ui/gui/spokes/network.glade.h:23
msgctxt "GUI|Network|Wireless"
msgid "_Network Name"
msgstr "Nomine del _rete"

#: pyanaconda/ui/gui/spokes/network.glade.h:24
msgid "Network Name"
msgstr "Nomine del rete"

#: pyanaconda/ui/gui/spokes/network.glade.h:25
msgid "Security Key"
msgstr "Clave de securitate"

#: pyanaconda/ui/gui/spokes/network.glade.h:26
msgctxt "GUI|Network|Wireless"
msgid "_Use as Hotspot..."
msgstr "_Usar como puncto de accesso..."

#: pyanaconda/ui/gui/spokes/network.glade.h:27
msgctxt "GUI|Network|Wireless"
msgid "_Stop Hotspot..."
msgstr "_Arrestar puncto de accesso..."

#: pyanaconda/ui/gui/spokes/network.glade.h:28
msgctxt "GUI|Network|Wireless"
msgid "_Configure..."
msgstr "_Configurar..."

#: pyanaconda/ui/gui/spokes/network.glade.h:29
msgid "IMEI"
msgstr "IMEI"

#: pyanaconda/ui/gui/spokes/network.glade.h:30
msgid "Provider"
msgstr "Fornitor"

#: pyanaconda/ui/gui/spokes/network.glade.h:32
msgctxt "GUI|Network|Mobile Broadband"
msgid "_Configure..."
msgstr "_Configurar..."

#: pyanaconda/ui/gui/spokes/network.glade.h:33
msgid "VPN Type"
msgstr "Typo de VPN"

#: pyanaconda/ui/gui/spokes/network.glade.h:34
msgid "Gateway"
msgstr "Passarella"

#: pyanaconda/ui/gui/spokes/network.glade.h:35
msgid "Group Name"
msgstr "Nomine de gruppo"

#: pyanaconda/ui/gui/spokes/network.glade.h:36
msgid "Group Password"
msgstr "Contrasigno de gruppo"

#: pyanaconda/ui/gui/spokes/network.glade.h:37
#: pyanaconda/ui/gui/spokes/user.glade.h:5
msgid "User name"
msgstr "Nomine de usator"

#: pyanaconda/ui/gui/spokes/network.glade.h:38
msgctxt "GUI|Network|VPN"
msgid "_Configure..."
msgstr "_Configurar..."

#: pyanaconda/ui/gui/spokes/network.glade.h:39
msgctxt "GUI|Network|Proxy"
msgid "_Method"
msgstr "_Methodo"

#: pyanaconda/ui/gui/spokes/network.glade.h:40
msgctxt "GUI|Network|Proxy"
msgid "_Configuration URL"
msgstr "URL de _configuration"

#: pyanaconda/ui/gui/spokes/network.glade.h:41
msgctxt "GUI|Network|Proxy"
msgid "HTTP _Proxy"
msgstr "_Proxy HTTP"

#: pyanaconda/ui/gui/spokes/network.glade.h:42
msgctxt "GUI|Network|Proxy"
msgid "H_TTPS Proxy"
msgstr "Proxy H_TTPS"

#: pyanaconda/ui/gui/spokes/network.glade.h:43
msgctxt "GUI|Network|Proxy"
msgid "_FTP Proxy"
msgstr "Proxy _FTP"

#: pyanaconda/ui/gui/spokes/network.glade.h:44
msgctxt "GUI|Network|Proxy"
msgid "_Socks Host"
msgstr "Host _SOCKS"

#: pyanaconda/ui/gui/spokes/network.glade.h:46
msgid "Network Config Box"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:47
msgctxt "GUI|Network"
msgid "Unloc_k"
msgstr "Disbloca_r"

#: pyanaconda/ui/gui/spokes/network.glade.h:48
msgctxt "GUI|Network"
msgid "Airplane Mode"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:49
msgid "More Network Config Box"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:50
msgctxt "GUI|Network"
msgid "_Host name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:52
msgid "_Apply"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:53
msgid "Set as current system host name."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:54
msgid "Current host name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade.h:55
msgid "NETWORK CONFIGURATION"
msgstr "CONFIGURATION DE RETE"

#: pyanaconda/ui/gui/spokes/network.glade.h:56
msgid ""
"We'll need network access to fetch information about your location and to "
"make software\n"
"updates available for you."
msgstr ""
"Le accesso al rete es necessari pro obtener le information super le location "
"e pro facer actualisationes de software disponibile."

#: pyanaconda/ui/gui/spokes/network.glade.h:58
msgid "Authentication"
msgstr "Authentication"

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#: pyanaconda/ui/gui/spokes/network.glade.h:59
msgctxt "GUI|Network|Authentication Dialog"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/network.glade.h:60
msgctxt "GUI|Network|Authentication Dialog"
msgid "C_onnect"
msgstr "C_onnecter"

#: pyanaconda/ui/gui/spokes/network.glade.h:62
msgid ""
"Passwords or encryption keys are required to access\n"
"the wireless network"
msgstr ""
"Contrasignos o claves cryptographic es necessari\n"
"pro connecter se al rete sin filo"

#: pyanaconda/ui/gui/spokes/software.glade.h:1
msgid "SOFTWARE SELECTION"
msgstr "SELECTION DE SOFTWARE"

#: pyanaconda/ui/gui/spokes/software.glade.h:2
msgid "Base Environment"
msgstr "Ambiente basic"

#: pyanaconda/ui/gui/spokes/software.glade.h:3
msgid "Add-Ons for Selected Environment"
msgstr "Additiones pro le ambiente seligite"

#: pyanaconda/ui/gui/spokes/storage.glade.h:1
msgctxt "GUI|Storage|Need Space Dialog"
msgid "Cancel & _add more disks"
msgstr "Annullar e _adder plus de discos"

# translation auto-copied from project anaconda, version f22-branch, document
# anaconda
#: pyanaconda/ui/gui/spokes/storage.glade.h:2
msgctxt "GUI|Storage|Need Space Dialog"
msgid "_Reclaim space"
msgstr "_Recuperar spatio"

#: pyanaconda/ui/gui/spokes/storage.glade.h:3
msgid "INSTALLATION OPTIONS"
msgstr "OPTIONES DE INSTALLATION"

#: pyanaconda/ui/gui/spokes/storage.glade.h:4
msgid ""
"Here we'll describe how much space is needed for the current software "
"selection."
msgstr ""
"Le amonta de spacio necessari pro installar programma seligite essera "
"monstrate hic."

#: pyanaconda/ui/gui/spokes/storage.glade.h:5
msgid "disk free"
msgstr "disco libere"

#: pyanaconda/ui/gui/spokes/storage.glade.h:6
msgid "Free space available for use."
msgstr "Spatio libere disponibile pro usage."

#: pyanaconda/ui/gui/spokes/storage.glade.h:7
msgid "fs free"
msgstr "systema de files libere"

#: pyanaconda/ui/gui/spokes/storage.glade.h:8
msgid "Free space unavailable but reclaimable from existing partitions."
msgstr ""
"Spatio libere que es indisponibile ma recuperabile del partitiones existente."

#: pyanaconda/ui/gui/spokes/storage.glade.h:9
msgid "Here we'll describe what your options are."
msgstr "Hic nos describera que es le optiones."

#: pyanaconda/ui/gui/spokes/storage.glade.h:10
msgid "Need Space"
msgstr ""

# translation auto-copied from project anaconda, version rhel7-branch,
# document anaconda
#: pyanaconda/ui/gui/spokes/storage.glade.h:11
msgctxt "GUI|Storage|No Space Dialog"
msgid "_Quit installer"
msgstr "_Quitar le installator"

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#: pyanaconda/ui/gui/spokes/storage.glade.h:12
msgctxt "GUI|Storage|No Space Dialog"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/storage.glade.h:13
msgid "No Space"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:15
msgid "Device Selection"
msgstr "Selection de dispositivo"

#: pyanaconda/ui/gui/spokes/storage.glade.h:16
msgid ""
"Select the device(s) you'd like to install to.  They will be left untouched "
"until you click on the main menu's \"Begin Installation\" button."
msgstr ""
"Selige le dispositivo(s) pro le installation. Illos non essera toccate usque "
"al clic sur le button «Comenciar le installation» in le menu principal."

#: pyanaconda/ui/gui/spokes/storage.glade.h:17
msgid "Local Standard Disks"
msgstr "Discos normal local"

#: pyanaconda/ui/gui/spokes/storage.glade.h:18
msgid "Disks left unselected here will not be touched."
msgstr "Le discos restate non seligite hic non essera toccate."

#: pyanaconda/ui/gui/spokes/storage.glade.h:19
msgid "Specialized & Network Disks"
msgstr "Discos specialisate e de rete"

#: pyanaconda/ui/gui/spokes/storage.glade.h:20
msgctxt "GUI|Storage"
msgid "_Add a disk..."
msgstr "_Adder un disco..."

#: pyanaconda/ui/gui/spokes/storage.glade.h:21
msgid "Add Specialized Disk"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:22
msgid "Other Storage Options"
msgstr "Altere optiones de immagazinage"

#: pyanaconda/ui/gui/spokes/storage.glade.h:23
msgid "Partitioning"
msgstr "Partitionamento"

#: pyanaconda/ui/gui/spokes/storage.glade.h:24
msgid "Encryption"
msgstr "Cryptation"

#: pyanaconda/ui/gui/spokes/storage.glade.h:25
msgid "A_utomatically configure partitioning."
msgstr "Configurar partitiones a_utomaticamente."

#: pyanaconda/ui/gui/spokes/storage.glade.h:26
msgid "_I will configure partitioning."
msgstr "Configurar partitiones _manualmente."

#: pyanaconda/ui/gui/spokes/storage.glade.h:27
msgid "_Encrypt my data."
msgstr "_Cifrar mi datos."

#: pyanaconda/ui/gui/spokes/storage.glade.h:28
msgid "You'll set a passphrase next."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:29
msgid "I would like to _make additional space available."
msgstr "Io vole _facer spatio additional disponibile."

#: pyanaconda/ui/gui/spokes/storage.glade.h:30
msgid "Storage Options"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:31
msgid "summary"
msgstr "summario"

#: pyanaconda/ui/gui/spokes/storage.glade.h:32
msgctxt "GUI|Storage"
msgid "_Full disk summary and boot loader..."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade.h:33
msgctxt "GUI|Storage"
msgid "_Refresh..."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:1
msgid "KEYBOARD LAYOUT"
msgstr "DISPOSITION DE CLAVIERO"

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:2
msgid ""
"Which keyboard layouts would you like to use on this system?  You may move "
"any layout to the top of the list to select it as the default."
msgstr ""
"Qual dispositiones de claviero debe usar se in iste systema? Move le "
"disposition in alto del lista pro usar lo per predefinition."

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:3
msgid ""
"Changes here will only apply to the installed system. Use the desktop's tool "
"to configure the keyboard for the installation process."
msgstr ""
"Cambiamentos hic affectara solmente le systema installate. Usa le "
"instrumentos de scriptorio pro configurar le claviero pro le processo de "
"installation."

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:4
msgid "name"
msgstr "nomine"

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:5
msgid "Selected Layouts"
msgstr "Dispositiones seligite"

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:6
msgctxt "GUI|Keyboard Layout"
msgid "_Add layout"
msgstr "_Adder disposition"

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:7
msgctxt "GUI|Keyboard Layout"
msgid "_Remove layout"
msgstr "_Remover disposition"

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:8
msgctxt "GUI|Keyboard Layout"
msgid "Move selected layout _up"
msgstr "Mover le disposition seligite in al_to"

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:9
msgctxt "GUI|Keyboard Layout"
msgid "Move selected layout _down"
msgstr "Mover le disposition seligite in _basso"

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:10
msgctxt "GUI|Keyboard Layout"
msgid "_Preview layout"
msgstr "_Previder disposition"

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:11
msgctxt "GUI|Keyboard Layout"
msgid "_Test the layout configuration below:"
msgstr "_Probar le disposition seligite in basso:"

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:12
msgid "Alt + Shift to switch layouts."
msgstr "Alt + Shift pro cambiar dispositiones."

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:13
msgctxt "GUI|Keyboard Layout"
msgid "_Options"
msgstr "_Optiones"

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#: pyanaconda/ui/gui/spokes/keyboard.glade.h:14
msgctxt "GUI|Keyboard Layout|Add Layout"
msgid "_Cancel"
msgstr "_Cancellar"

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#: pyanaconda/ui/gui/spokes/keyboard.glade.h:15
msgctxt "GUI|Keyboard Layout|Add Layout"
msgid "_Add"
msgstr "_Adder"

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:16
msgid "ADD A KEYBOARD LAYOUT"
msgstr "ADDER UN DISPOSITION DE CLAVIERO"

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:17
msgid "You may add a keyboard layout by selecting it below:"
msgstr ""
"Le disposition de claviero pote esser addite per le selection in basso:"

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:19
msgid "Available Layouts"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:20
msgid "Add Layout"
msgstr "Adder disposition"

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#: pyanaconda/ui/gui/spokes/keyboard.glade.h:21
msgctxt "GUI|Keyboard Layout|Switching Options"
msgid "_Cancel"
msgstr "_Cancellar"

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#: pyanaconda/ui/gui/spokes/keyboard.glade.h:22
msgctxt "GUI|Keyboard Layout|Switching Options"
msgid "_OK"
msgstr "_OK"

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:23
msgid "LAYOUT SWITCHING OPTIONS"
msgstr "OPTIONES DEL CAMBIAMENTO DE DISPOSITION"

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:24
msgid ""
"Which combination(s) would you prefer for switching between keyboard layouts?"
msgstr ""
"Qual combination(es) de claves debe usar se pro cambiar dispositiones de "
"clavieros?"

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:25
msgid "use"
msgstr "usar"

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:26
msgid "description"
msgstr "description"

#: pyanaconda/ui/gui/spokes/keyboard.glade.h:27
msgid "Layout Options"
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.glade.h:1
msgid "LANGUAGE SUPPORT"
msgstr "SUPPORTO DE LINGUAS"

#: pyanaconda/ui/gui/spokes/langsupport.glade.h:2
msgid "Select additional language support to be installed:"
msgstr "Selige le supporto pro linguas additional que debe esser installate:"

#: pyanaconda/ui/gui/spokes/langsupport.glade.h:3
#: pyanaconda/ui/gui/spokes/welcome.glade.h:11
msgid "nativeName"
msgstr "Nomine native"

#: pyanaconda/ui/gui/spokes/langsupport.glade.h:4
#: pyanaconda/ui/gui/spokes/welcome.glade.h:12
msgid "englishName"
msgstr "Lingua"

#: pyanaconda/ui/gui/spokes/langsupport.glade.h:5
#: pyanaconda/ui/gui/spokes/welcome.glade.h:13
msgid "selected"
msgstr "seligite"

#: pyanaconda/ui/gui/spokes/langsupport.glade.h:6
#: pyanaconda/ui/gui/spokes/welcome.glade.h:15
msgid "Type here to search."
msgstr "Scribe hic pro cercar."

#: pyanaconda/ui/gui/spokes/langsupport.glade.h:7
msgid "checked"
msgstr "verificate"

#: pyanaconda/ui/gui/spokes/password.glade.h:1
msgid "ROOT PASSWORD"
msgstr "CONTRASIGNO DE SUPERUSATOR"

#: pyanaconda/ui/gui/spokes/password.glade.h:2
msgctxt "GUI|Password"
msgid "_Root Password:"
msgstr "Contrasigno del supe_rusator:"

#: pyanaconda/ui/gui/spokes/password.glade.h:3
msgctxt "GUI|Password"
msgid "_Confirm:"
msgstr "_Confirmar:"

#: pyanaconda/ui/gui/spokes/password.glade.h:5
#: pyanaconda/ui/gui/spokes/user.glade.h:9
msgid "Confirm Password"
msgstr "Confirmar contrasigno"

#: pyanaconda/ui/gui/spokes/password.glade.h:6
#: pyanaconda/ui/gui/spokes/user.glade.h:12
msgid "empty password"
msgstr "contrasigno vacue"

#: pyanaconda/ui/gui/spokes/password.glade.h:7
msgid ""
"The root account is used for administering the system.  Enter a password for "
"the root user."
msgstr ""
"Le conto del superusator root se utilisa pro administrar le systema. "
"Introduce un contrasigno pro le usator root."

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#: pyanaconda/ui/gui/spokes/source.glade.h:1
msgctxt "GUI|Software Source|ISO Chooser Dialog"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/source.glade.h:2
msgctxt "GUI|Software Source|ISO Chooser Dialog"
msgid "_Open"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:3
msgctxt "GUI|Software Source|Media Check Dialog"
msgid "_Done"
msgstr "Pr_este"

#: pyanaconda/ui/gui/spokes/source.glade.h:4
msgid "MEDIA VERIFICATION"
msgstr "VERIFICATION DE MEDIO"

#: pyanaconda/ui/gui/spokes/source.glade.h:5
msgid "Verifying media, please wait..."
msgstr "Verification del medio, attende..."

#: pyanaconda/ui/gui/spokes/source.glade.h:6
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Cancel"
msgstr "_Cancellar"

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#: pyanaconda/ui/gui/spokes/source.glade.h:7
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_OK"
msgstr "_OK"

#: pyanaconda/ui/gui/spokes/source.glade.h:8
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Enable HTTP Proxy"
msgstr "Activar proxy _HTTP"

#: pyanaconda/ui/gui/spokes/source.glade.h:9
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Proxy Host"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:10
msgid "<span size=\"small\"><b>Example:</b> squid.mysite.org:3128</span>"
msgstr "<span size=\"small\"><b>Exemplo:</b> squid.mysite.org:3128</span>"

#: pyanaconda/ui/gui/spokes/source.glade.h:11
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Use Authentication"
msgstr "_Usar authentication"

#: pyanaconda/ui/gui/spokes/source.glade.h:12
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "User _name"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:13
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "Pass_word"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:14
msgid "INSTALLATION SOURCE"
msgstr "FONTE DE INSTALLATION"

#: pyanaconda/ui/gui/spokes/source.glade.h:15
msgid "Which installation source would you like to use?"
msgstr "Qual fonte de installation debe usar se?"

#: pyanaconda/ui/gui/spokes/source.glade.h:16
msgctxt "GUI|Software Source"
msgid "_Auto-detected installation media:"
msgstr "Medio de installation _autodetegite:"

#: pyanaconda/ui/gui/spokes/source.glade.h:17
msgid "Device:"
msgstr "Dispositivo:"

#: pyanaconda/ui/gui/spokes/source.glade.h:18
msgid "Label:"
msgstr "Etiquetta:"

#: pyanaconda/ui/gui/spokes/source.glade.h:19
msgctxt "GUI|Software Source"
msgid "_Verify"
msgstr "_Verificar"

#: pyanaconda/ui/gui/spokes/source.glade.h:20
msgctxt "GUI|Software Source"
msgid "Installation media via SE/_HMC"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:21
msgctxt "GUI|Software Source"
msgid "_ISO file:"
msgstr "File _ISO:"

#: pyanaconda/ui/gui/spokes/source.glade.h:22
msgctxt "GUI|Software Source"
msgid "D_evice:"
msgstr "D_ispositivo:"

#: pyanaconda/ui/gui/spokes/source.glade.h:23
msgctxt "GUI|Software Source"
msgid "_Choose an ISO"
msgstr "_Eliger un ISO"

#: pyanaconda/ui/gui/spokes/source.glade.h:24
msgctxt "GUI|Software Source"
msgid "_On the network:"
msgstr "_In le rete:"

#: pyanaconda/ui/gui/spokes/source.glade.h:26
msgctxt "GUI|Software Source"
msgid "_Proxy setup..."
msgstr "C_onfiguration del proxy..."

#: pyanaconda/ui/gui/spokes/source.glade.h:27
msgctxt "GUI|Software Source"
msgid "This URL refers to a _mirror list."
msgstr "Iste URL se refere a un lista de specu_los."

#: pyanaconda/ui/gui/spokes/source.glade.h:28
msgctxt "GUI|Software Source"
msgid "N_FS mount options:"
msgstr "Optiones de montage N_FS:"

#: pyanaconda/ui/gui/spokes/source.glade.h:29
msgid "This field is optional."
msgstr "Iste campo es optional."

#: pyanaconda/ui/gui/spokes/source.glade.h:30
msgid "Updates"
msgstr "Actualisationes"

#: pyanaconda/ui/gui/spokes/source.glade.h:31
msgctxt "GUI|Software Source"
msgid ""
"Don't install the latest available software _updates. Install the default "
"versions provided by the installation source above."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:32
msgid "Additional repositories"
msgstr "Repositorios additional"

#: pyanaconda/ui/gui/spokes/source.glade.h:33
msgid "Enabled"
msgstr "Activate"

#: pyanaconda/ui/gui/spokes/source.glade.h:35
msgid "Add a new repository."
msgstr "Adder un nove repositorio."

#: pyanaconda/ui/gui/spokes/source.glade.h:36
msgctxt "GUI|Software Source"
msgid "A_dd"
msgstr "A_dder"

#: pyanaconda/ui/gui/spokes/source.glade.h:37
msgid "Remove the selected repository."
msgstr "Remover le repositorio seligite."

#: pyanaconda/ui/gui/spokes/source.glade.h:38
msgctxt "GUI|Software Source"
msgid "_Remove"
msgstr "_Remover"

#: pyanaconda/ui/gui/spokes/source.glade.h:39
msgid "Revert to the previous list of repositories."
msgstr "Revenir al previe lista de repositorios."

#: pyanaconda/ui/gui/spokes/source.glade.h:40
msgctxt "GUI|Software Source"
msgid "Rese_t"
msgstr "Reini_tialisar"

#: pyanaconda/ui/gui/spokes/source.glade.h:41
msgid "URL for the repository, without protocol."
msgstr "URL pro le repositorio, sin protocollo."

#: pyanaconda/ui/gui/spokes/source.glade.h:42
msgid "URL of proxy in the form of protocol://host:[port]"
msgstr "URL de proxy in le forma protocollo://host:[porto]"

#: pyanaconda/ui/gui/spokes/source.glade.h:43
msgid "Optional proxy user name."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:44
msgid "Optional proxy password."
msgstr "Contrasigno optional al proxy."

#: pyanaconda/ui/gui/spokes/source.glade.h:45
msgctxt "GUI|Software Source"
msgid "This URL refers to a mirror _list."
msgstr "Iste URL se refere a un _lista de speculos."

#: pyanaconda/ui/gui/spokes/source.glade.h:46
msgctxt "GUI|Software Source"
msgid "_Name:"
msgstr "No_mine:"

#: pyanaconda/ui/gui/spokes/source.glade.h:47
msgid "Protocol for the repository URL."
msgstr "Protocollo pro le URL de repositorio."

#: pyanaconda/ui/gui/spokes/source.glade.h:48
msgctxt "GUI|Software Source"
msgid "Pro_xy URL:"
msgstr "URL de pro_xy:"

#: pyanaconda/ui/gui/spokes/source.glade.h:49
msgctxt "GUI|Software Source"
msgid "U_ser name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade.h:50
msgctxt "GUI|Software Source"
msgid "Pass_word:"
msgstr "Contra_signo:"

#: pyanaconda/ui/gui/spokes/source.glade.h:51
msgid "Name of the repository."
msgstr "Nomine del repositorio."

#: pyanaconda/ui/gui/spokes/welcome.glade.h:1
msgctxt "GUI|Welcome|Beta Warn Dialog"
msgid "I want to _exit."
msgstr "Io vole _sortir."

#: pyanaconda/ui/gui/spokes/welcome.glade.h:2
msgctxt "GUI|Welcome|Beta Warn Dialog"
msgid "I want to _proceed."
msgstr "Io vole _proceder."

#: pyanaconda/ui/gui/spokes/welcome.glade.h:3
msgid "This is unstable, pre-release software."
msgstr "Iste software es instabile, in stato de pre-edition."

#: pyanaconda/ui/gui/spokes/welcome.glade.h:4
msgid ""
"Notice:  This is pre-release software that is intended for development and "
"testing purposes only.  Do not use this software for any critical work or "
"for production environments.\n"
"\n"
"By clicking \"I want to proceed\", you understand and accept the risks "
"associated with pre-release software, that you intend to use this for "
"testing and development purposes only, and are willing to report any bugs or "
"issues in order to enhance this work.\n"
"\n"
"If you do not understand or accept the risks, then please exit this program "
"by clicking \"I want to exit\" which will reboot your system."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade.h:9
msgid "Beta Warn"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade.h:10
msgid "What language would you like to use during the installation process?"
msgstr "Que lingua debe esser usate durante le processo de installation?"

# translation auto-copied from project comps, version master, document comps
#: pyanaconda/ui/gui/spokes/welcome.glade.h:14
msgid "Languages"
msgstr "Linguas"

#: pyanaconda/ui/gui/spokes/welcome.glade.h:16
msgid "Locales"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade.h:17
msgid "WELCOME"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade.h:18
msgctxt "GUI|Welcome|Unsupported Hardware Dialog"
msgid "_Quit"
msgstr "_Quitar"

#: pyanaconda/ui/gui/spokes/welcome.glade.h:19
msgctxt "GUI|Welcome|Unsupported Hardware Dialog"
msgid "_Continue"
msgstr "_Continuar"

#: pyanaconda/ui/gui/spokes/welcome.glade.h:20
msgid "Unsupported Hardware Detected"
msgstr "Hardware non supportate detegite"

#: pyanaconda/ui/gui/spokes/welcome.glade.h:22
msgid "Unsupported Hardware"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade.h:1
msgid "CREATE USER"
msgstr "CREAR UN USATOR"

#: pyanaconda/ui/gui/spokes/user.glade.h:2
msgctxt "GUI|User"
msgid "_Full name"
msgstr "No_mine complete"

#: pyanaconda/ui/gui/spokes/user.glade.h:3
msgctxt "GUI|User"
msgid "_User name"
msgstr "Nomine de _usator"

#: pyanaconda/ui/gui/spokes/user.glade.h:4
msgid "Full Name"
msgstr "Nomine complete"

#: pyanaconda/ui/gui/spokes/user.glade.h:6
msgctxt "GUI|User"
msgid "_Password"
msgstr "Contra_signo"

#: pyanaconda/ui/gui/spokes/user.glade.h:7
msgctxt "GUI|User"
msgid "_Confirm password"
msgstr "_Confirmar le contrasigno"

#: pyanaconda/ui/gui/spokes/user.glade.h:10
msgid ""
"<b>Tip:</b> Keep your user name shorter than 32 characters and do not use "
"spaces."
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade.h:11
msgctxt "GUI|User"
msgid "Require a password to use this account"
msgstr "Requirer un contrasigno pro usar iste conto"

#: pyanaconda/ui/gui/spokes/user.glade.h:13
msgctxt "GUI|User"
msgid "Make this user administrator"
msgstr "Facer le administrator de iste usator"

#: pyanaconda/ui/gui/spokes/user.glade.h:14
msgctxt "GUI|User"
msgid "_Advanced..."
msgstr "_Avantiate..."

#: pyanaconda/ui/gui/spokes/custom.glade.h:6
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:23
msgid "RAID0 <span foreground=\"grey\">(Performance)</span>"
msgstr "RAID0 <span foreground=\"grey\">(Rapiditate)</span>"

#: pyanaconda/ui/gui/spokes/custom.glade.h:7
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:24
msgid "RAID1 <span foreground=\"grey\">(Redundancy)</span>"
msgstr "RAID1 <span foreground=\"grey\">(Redundantia)</span>"

#: pyanaconda/ui/gui/spokes/custom.glade.h:8
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:25
msgid "RAID4 <span foreground=\"grey\">(Error Checking)</span>"
msgstr "RAID4 <span foreground=\"grey\">(Controlo de errores)</span>"

#: pyanaconda/ui/gui/spokes/custom.glade.h:9
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:26
msgid "RAID5 <span foreground=\"grey\">(Distributed Error Checking)</span>"
msgstr ""
"RAID5 <span foreground=\"grey\">(Controlo de errores distribute)</span>"

#: pyanaconda/ui/gui/spokes/custom.glade.h:10
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:27
msgid "RAID6 <span foreground=\"grey\">(Redundant Error Checking)</span>"
msgstr ""
"RAID6 <span foreground=\"grey\">(Controlo de errores redundante)</span>"

#: pyanaconda/ui/gui/spokes/custom.glade.h:11
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:28
msgid "RAID10 <span foreground=\"grey\">(Performance, Redundancy)</span>"
msgstr "RAID10 <span foreground=\"grey\">(Rapiditate, redundantia)</span>"

#: pyanaconda/ui/gui/spokes/custom.glade.h:13
msgid "Add a new mount point."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:14
msgctxt "GUI|Custom Partitioning"
msgid "Add"
msgstr "Adder"

#: pyanaconda/ui/gui/spokes/custom.glade.h:15
msgid "Remove the selected mount point(s)."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:16
msgctxt "GUI|Custom Partitioning"
msgid "Remove"
msgstr "Remover"

#: pyanaconda/ui/gui/spokes/custom.glade.h:17
msgid "Reload storage configuration from disk."
msgstr "Recargar le configuration de immagazinage del disco."

#: pyanaconda/ui/gui/spokes/custom.glade.h:18
msgctxt "GUI|Custom Partitioning"
msgid "Refresh"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:19
msgid "Selected Device"
msgstr "Dispositivo seligite"

#: pyanaconda/ui/gui/spokes/custom.glade.h:20
msgid "Device description"
msgstr "Description de dispositivo"

#: pyanaconda/ui/gui/spokes/custom.glade.h:21
msgid "Mount _Point:"
msgstr "_Puncto de montage:"

#: pyanaconda/ui/gui/spokes/custom.glade.h:22
msgid "_Desired Capacity:"
msgstr "Capacitate _desirabile:"

#: pyanaconda/ui/gui/spokes/custom.glade.h:23
msgid "_Label:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:24
msgid "Device(s):"
msgstr ""

# translation auto-copied from project anaconda, version f22-branch, document
# anaconda
#: pyanaconda/ui/gui/spokes/custom.glade.h:25
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "_Modify..."
msgstr "_Modificar…"

#: pyanaconda/ui/gui/spokes/custom.glade.h:26
msgid "_Name:"
msgstr "No_mine:"

#: pyanaconda/ui/gui/spokes/custom.glade.h:27
msgctxt "GUI|Custom Partitioning|Configure"
msgid "Device _Type:"
msgstr "_Typo de dispositivo:"

#: pyanaconda/ui/gui/spokes/custom.glade.h:28
msgctxt "GUI|Custom Partitioning|Configure"
msgid "_Encrypt"
msgstr "_Cryptar"

#: pyanaconda/ui/gui/spokes/custom.glade.h:29
msgctxt "GUI|Custom Partitioning|Configure"
msgid "File S_ystem:"
msgstr "S_ystema de files:"

#: pyanaconda/ui/gui/spokes/custom.glade.h:30
msgctxt "GUI|Custom Partitioning|Configure"
msgid "Ref_ormat"
msgstr "Re_formatar"

#: pyanaconda/ui/gui/spokes/custom.glade.h:31
msgctxt "GUI|Custom Partitioning|Configure"
msgid "_Volume Group:"
msgstr "Gruppo de _volumines:"

#: pyanaconda/ui/gui/spokes/custom.glade.h:32
msgctxt "GUI|Custom Partitioning|Configure"
msgid "_Modify..."
msgstr "_Modificar..."

#: pyanaconda/ui/gui/spokes/custom.glade.h:33
msgctxt "GUI|Custom Partitioning|Configure"
msgid "RA_ID Level:"
msgstr "Nivello de RA_ID:"

#: pyanaconda/ui/gui/spokes/custom.glade.h:34
msgid "_Update Settings"
msgstr "Parametros de act_ualisation"

#: pyanaconda/ui/gui/spokes/custom.glade.h:35
msgid ""
"Note:  The settings you make on this screen will not be applied until you "
"click on the main menu's 'Begin Installation' button."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade.h:36
msgid ""
"This device is encrypted and cannot be read without a valid passphrase. You "
"may unlock it below."
msgstr ""
"Iste dispositivo es cryptate e non pote esser legite sin un phrase de "
"contrasigno valide. Tu pote disblocar lo in basso."

#: pyanaconda/ui/gui/spokes/custom.glade.h:37
msgctxt "GUI|Custom Partitioning|Encrypted"
msgid "_Passphrase:"
msgstr "_Phrase de contrasigno:"

#: pyanaconda/ui/gui/spokes/custom.glade.h:38
msgctxt "GUI|Custom Partitioning|Encrypted"
msgid "_Unlock"
msgstr "_Disblocar"

#: pyanaconda/ui/gui/spokes/custom.glade.h:39
msgid ""
"This device cannot be edited directly. You can remove it or select a "
"different device."
msgstr ""
"Iste dispositivo non pote esser editate directemente. On pote remover lo o "
"seliger un altere dispositivo."

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/ui/gui/spokes/custom.glade.h:40
msgid "AVAILABLE SPACE"
msgstr "SPATIO DISPONIBILE"

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/ui/gui/spokes/custom.glade.h:41
msgid "TOTAL SPACE"
msgstr "SPATIO TOTAL"

#: pyanaconda/ui/gui/spokes/custom.glade.h:42
msgctxt "GUI|Custom Partitioning"
msgid "_Reset All"
msgstr "_Reinitialisar toto"

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade.h:1
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Cancel"
msgstr "_Cancellar"

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade.h:2
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_OK"
msgstr "_OK"

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade.h:3
msgid "ADD DASD STORAGE TARGET"
msgstr "ADDER SCOPO DE IMMAGAZINAGE DASD"

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade.h:4
msgid "To use DASD disks, you must provide the device number."
msgstr ""

# translation auto-copied from project anaconda, version rhel7-branch,
# document anaconda
#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade.h:5
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Device number:"
msgstr "Numero de _dispositivo:"

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade.h:6
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Start Discovery"
msgstr "_Initiar discoperta"

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade.h:7
msgid "Discovering DASD devices.  This may take a moment..."
msgstr "Discoperimento de dispositivos DASD. Isto pote prender un momento…"

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade.h:8
#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade.h:7
msgid "Device discovery failed."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade.h:9
msgid ""
"The following error occurred discovering DASD devices.  Please double check "
"your configuration information and try again."
msgstr ""
"Le sequente error occurreva durante le discoperimento del dispositivos DASD. "
"Verifica le information de configuration e reproba."

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade.h:10
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Retry Discovery"
msgstr "_Reprobar discoperta"

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade.h:11
msgid "Device discovery succeeded."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade.h:12
msgid "The added DASD can be found under \"Local Standard Disks\"."
msgstr ""

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade.h:1
msgctxt "GUI|Advanced Storage|FCoE"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade.h:2
msgid ""
"Please select the network interface which is connected to your FCoE switch."
msgstr "Selige le interfacie de rete que es connectite al commutator FCoE."

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade.h:3
msgctxt "GUI|Advanced Storage|FCoE"
msgid "_NIC:"
msgstr "_NIC:"

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade.h:4
msgctxt "GUI|Advanced Storage|FCoE"
msgid "Use _DCB"
msgstr "Usar _DCB"

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade.h:5
msgctxt "GUI|Advanced Storage|FCoE"
msgid "Use auto _VLAN"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade.h:6
msgctxt "GUI|Advanced Storage|FCoE"
msgid "_Add FCoE Disk(s)"
msgstr "_Adder disco(s) FCoE"

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade.h:7
msgid "Error adding FCoE SAN."
msgstr "Error al addition de SAN FCoE."

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade.h:8
msgid ""
"The following error occurred while attempting to add an FCoE SAN.  Please "
"check your configuration and try again"
msgstr ""
"Le sequente error occurreva durante le tentativa de adder un SAN FCoE. "
"Verifica le configuration e reproba"

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:1
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Cancel"
msgstr "_Cancellar"

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:2
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_OK"
msgstr "_OK"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:3
msgid "ADD iSCSI STORAGE TARGET"
msgstr "ADDER SCOPO DE IMMAGAZINAGE iSCSI"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:4
msgid ""
"To use iSCSI disks, you must provide the address of your iSCSI target and "
"the iSCSI initiator name you've configured for your host."
msgstr ""
"Pro usar discos iSCSI, on debe fornir le adresse de scopo iSCSI e le nomine "
"de initiator iSCSI configurate pro le host."

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:5
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Target IP Address:"
msgstr "Adresse IP del _scopo:"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:6
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "iSCSI _Initiator Name:"
msgstr "Nomine de _initiator iSCSI:"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:7
msgid "No credentials (discovery authentication disabled)"
msgstr "Nulle datos de authentication (discoperta es disactivate)"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:8
msgid "CHAP pair"
msgstr "Par de CHAP"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:9
msgid "CHAP pair and a reverse pair"
msgstr "Par de CHAP e un par reverse"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:10
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Discovery Authentication Type:"
msgstr "Typo de authentication de _discoperimento:"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:11
msgid ""
"<span size=\"small\"><b>Example:</b> iqn.2012-09.com.example:diskarrays-sn-"
"a8675309</span>"
msgstr ""
"<span size=\"small\"><b>Exemplo:</b> iqn.2012-09.com.example:diskarrays-sn-"
"a8675309</span>"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:12
msgctxt "GUI|Advanced Storage|iSCSI|Configure|CHAP"
msgid "CHAP _User name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:13
msgctxt "GUI|Advanced Storage|iSCSI|Configure|CHAP"
msgid "CHAP _Password:"
msgstr "Contra_signo CHAP:"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:14
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "CHAP _User name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:15
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "CHAP _Password:"
msgstr "Contra_signo CHAP:"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:16
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "Reverse CHAP User _name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:17
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "Reverse CHAP Pass_word:"
msgstr "_Contrasigno CHAP reverse:"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:18
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Start"
msgid "_Start Discovery"
msgstr "_Initiar discoperimento"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:19
msgid "Discovering iSCSI targets.  This may take a moment..."
msgstr "Discoperimento de scopos iSCSI. Isto pote prender un momento…"

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:20
msgid "Discovery login rejected."
msgstr "Insuccesso de authentication al discoperimento."

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:21
msgid ""
"The following error occurred discovering iSCSI targets.  Please double check "
"your authorization information and try again."
msgstr ""
"Le sequente error occurreva durante le discoperimento del scopos iSCSI. "
"Verifica le information de authentication e reproba."

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:22
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Retry"
msgid "_Retry Discovery"
msgstr "_Tentar le discoperimento de novo"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:23
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Bind targets to network interfaces. This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:24
msgid "Node Name"
msgstr "Nomine de nodo"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:25
msgid "Interface"
msgstr "Interfacie"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:26
msgid "Portal"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:27
msgctxt "GUI|Advanced Storage|iSCSI|Login"
msgid "_Node Login Authentication Type:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:28
msgid "Use the credentials from discovery"
msgstr "Usar le datos de authentication desde discoperta"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:29
msgctxt "GUI|Advanced Storage|iSCSI|Login|CHAP"
msgid "CHAP _User name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:30
msgctxt "GUI|Advanced Storage|iSCSI|Login|CHAP"
msgid "CHAP _Password:"
msgstr "Contra_signo CHAP:"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:31
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "CHAP _User name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:32
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "CHAP _Password:"
msgstr "Contra_signo CHAP:"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:33
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "_Reverse CHAP User name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:34
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "Reverse CHAP Pass_word:"
msgstr "_Contrasigno CHAP reverse:"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:35
msgctxt "GUI|Advanced Storage|iSCSI|Login|Login"
msgid "_Log In"
msgstr "_Authenticar se"

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:36
msgid "Node login failed."
msgstr "Impossibile de authenticar se al nodo."

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:37
msgid ""
"The following error occurred logging into the selected iSCSI node.  Please "
"double check your authorization information and try again"
msgstr ""
"Le sequente error ha occurrite durante de authentication al nodo iSCSI. "
"Verifica le information de authentication e tenta de novo."

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade.h:38
msgctxt "GUI|Advanced Storage|iSCSI|Login|Retry"
msgid "Retry _Log In"
msgstr "_Authenticar se de novo"

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade.h:1
msgctxt "GUI|Advanced Storage|NVDIMM|Reconfigure"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade.h:2
msgid ""
"The following NVDIMM devices selected from the list will be reconfigured to "
"sector mode:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade.h:3
msgctxt "GUI|Advanced Storage|NVDIMM|Reconfigure"
msgid "_Sector size:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade.h:4
msgctxt "GUI|Advanced Storage|NVDIMM|Reconfigure"
msgid "Start _Reconfiguration"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade.h:5
msgid "Reconfiguring NVDIMM devices. This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade.h:6
msgid "Device reconfiguration failed."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade.h:7
msgid "The following error occurred reconfiguring NVDIMM devices:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade.h:8
msgid "Devices reconfiguration succeeded."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade.h:9
msgid ""
"Rescanning disks after reconfiguration.\n"
"This may take a moment ..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade.h:11
msgid "Warning: Existing data on reconfigured devices will be lost."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade.h:1
msgid "ADD zFCP STORAGE TARGET"
msgstr "ADDER SCOPO DE IMMAGAZINAGE zFCP"

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade.h:2
msgid ""
"To use zFCP disks, you must provide the device number, WWPN, and LUN "
"configured for the device."
msgstr ""
"Pro utilisar discos zFCP, on debe fornir le numero de dispositivo, WWPN e "
"LUN que es configurate pro le dispositivo."

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade.h:3
msgctxt "GUI|Advanced Storage|zFCP|Device Number"
msgid "_Device number:"
msgstr "Numero de _dispositivo:"

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade.h:4
msgid "WWPN:"
msgstr "WWPN:"

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade.h:5
msgctxt "GUI|Advanced Storage|zFCP|Start Discovery"
msgid "_Start Discovery"
msgstr "_Initiar discoperta"

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade.h:6
msgid "Discovering zFCP devices.  This may take a moment..."
msgstr "Discoperimento de dispositivos zFCP. Isto pote prender un momento…"

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade.h:8
msgid ""
"The following error occurred discovering zFCP devices.  Please double check "
"your configuration information and try again."
msgstr ""
"Le sequente error occurreva durante le discoperimento del dispositivos zFCP. "
"Verifica le information de configuration e reproba."

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade.h:9
msgctxt "GUI|Advanced Storage|zFCP|Retry Discovery"
msgid "_Retry Discovery"
msgstr "_Reprobar discoperta"

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade.h:10
msgid "LUN:"
msgstr "LUN:"

#: pyanaconda/ui/gui/spokes/lib/cart.glade.h:1
msgid "SELECTED DISKS"
msgstr "DISCOS SELIGITE"

#: pyanaconda/ui/gui/spokes/lib/cart.glade.h:2
msgctxt "GUI|Selected Disks Dialog"
msgid "_Close"
msgstr "_Clauder"

#: pyanaconda/ui/gui/spokes/lib/cart.glade.h:3
msgid "Boot"
msgstr "Boot"

#: pyanaconda/ui/gui/spokes/lib/cart.glade.h:4
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:7
msgid "Description"
msgstr "Description"

#: pyanaconda/ui/gui/spokes/lib/cart.glade.h:7
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:10
msgid "Free"
msgstr "Libere"

#: pyanaconda/ui/gui/spokes/lib/cart.glade.h:9
msgctxt "GUI|Selected Disks Dialog"
msgid "_Remove"
msgstr "_Remover"

#: pyanaconda/ui/gui/spokes/lib/cart.glade.h:10
msgid "Disk summary goes here"
msgstr "Le summario de disco es hic"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:1
msgctxt "GUI|Custom Partitioning|Confirm Delete Dialog"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:2
msgctxt "GUI|Custom Partitioning|Confirm Delete Dialog"
msgid "_Delete It"
msgstr "_Deler lo"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:3
msgctxt "GUI|Custom Partitioning|Configure Dialog"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:4
msgctxt "GUI|Custom Partitioning|Configure Dialog"
msgid "_Select"
msgstr "_Seliger"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:5
msgid "CONFIGURE MOUNT POINT"
msgstr "CONFIGURAR PUNCTOS DE MONTAGE"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:6
msgid "Select one or more disks this device may reside on."
msgstr "Selige un o alicun discos ubi iste dispositivo pote resider."

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:11
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:12
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "_Add mount point"
msgstr "Adder le _puncto de montage"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:13
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "_Desired Capacity:"
msgstr "Capacitate _desirabile:"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:14
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "Mount _Point:"
msgstr "_Puncto de montage:"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:15
msgid "ADD A NEW MOUNT POINT"
msgstr "ADDER UN NOVE PUNCTO DE MONTAGE"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:16
msgid ""
"More customization options are available\n"
"after creating the mount point below."
msgstr ""
"Plus del optiones de personalisation es disponibile\n"
"post creation del puncto de montage in basso."

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:18
msgid ""
"That mount point is already in\n"
"use.  Try something else?"
msgstr ""
"Iste puncto de montage ja es \n"
"usate. Probar un altere cosa?"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:20
msgid "eg: \"20 GB\", \"500mb\" (minus the quotation marks)"
msgstr "e.g.: \"20 GB\", \"500mb\" (minus le virgulettas)"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:22
msgid "Single <span foreground=\"grey\">(No Redundancy, No Striping)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:29
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:30
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "_Save"
msgstr "_Salveguardar"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:31
msgid "CONFIGURE CONTAINER"
msgstr "CONFIGURAR LE CONTAINER"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:32
msgid ""
"Please create a name for this container and select at least one disk below."
msgstr "Crea un nomine pro iste container e selige al minus un disco in basso."

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:33
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "_Name:"
msgstr "No_mine:"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:34
msgid "RAID Level:"
msgstr "Nivello de RAID:"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:35
msgid "Encrypt"
msgstr "Cryptar"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:36
msgid "Please enter a valid name."
msgstr "Introduce un nomine valide."

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:37
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "Si_ze policy:"
msgstr "Regulamento de _dimension:"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:38
msgid "Automatic"
msgstr "Automatic"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:39
msgid "As large as possible"
msgstr "Le plus grande possibile"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade.h:40
msgid "Fixed"
msgstr "Fixate"

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade.h:1
msgid "UNFORMATTED DASDS"
msgstr "DASDS NON FORMATATE"

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade.h:2
msgctxt "GUI|DASD Format Dialog"
msgid "_Cancel"
msgstr "A_nnullar"

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade.h:3
msgctxt "GUI|DASD Format Dialog"
msgid "_OK"
msgstr "_OK"

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade.h:4
msgid ""
"The following unformatted or LDL DASDs have been detected on your system. "
"You can choose to format them now with dasdfmt or cancel to leave them "
"unformatted. Unformatted DASDs cannot be used during installation."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade.h:5
msgid "Unformatted DASDs Detected"
msgstr "Discoperite DASDs sin formatation"

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade.h:6
msgid ""
"<b>Warning</b>: All storage changes made using the installer will be lost "
"when you choose to format."
msgstr ""
"<b>Aviso</b>: Tote le cambiamentos de immagazinage facite per le uso de "
"installator essera perdite quando on elige formatar."

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade.h:7
msgctxt "GUI|DASD Format Dialog"
msgid "_Format with dasdfmt"
msgstr "_Format con dasfmt"

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade.h:8
msgid ""
"You may <a href=\"\">go back to the main menu</a> to complete other\n"
"installation options while formatting completes."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade.h:10
msgid "Disk formatting complete."
msgstr "Formatation de disco complete."

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade.h:11
#: pyanaconda/ui/gui/spokes/lib/refresh.glade.h:11
msgid ""
"Pressing 'OK' below will take you to the disk selection screen where you "
"will need to re-select your disks."
msgstr ""
"Pressar sur 'OK' in basso va transferer al schermo a seliger discos ubi "
"illos debe esser seligite de novo."

#: pyanaconda/ui/gui/spokes/lib/detailederror.glade.h:1
msgid "An unknown error occurred during installation.  Details are below."
msgstr ""
"Un error incognite ha occurrite durante le installation. Plus de detalios es "
"in basso."

#: pyanaconda/ui/gui/spokes/lib/entropy_dialog.glade.h:1
msgid ""
"The system needs better quality of random data, you can improve it by typing "
"randomly on keyboard and moving your mouse. The installation will continue "
"automatically regardless of random data quality when time runs out."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/entropy_dialog.glade.h:2
msgid "Random data quality:"
msgstr ""

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#: pyanaconda/ui/gui/spokes/lib/passphrase.glade.h:1
msgctxt "GUI|Passphrase Dialog"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade.h:2
msgctxt "GUI|Passphrase Dialog"
msgid "_Save Passphrase"
msgstr "_Salveguardar le phrase de contrasigno"

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade.h:3
msgid "DISK ENCRYPTION PASSPHRASE"
msgstr "PHRASE DE CONTRASIGNO PRO CRYPTAR LE DISCO"

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade.h:4
msgid ""
"You have chosen to encrypt some of your data. You will need to create a "
"passphrase that you will use to access your data when you start your "
"computer."
msgstr ""
"Tu ha eligite a cryptar certe datos. Tu debe crear un phrase de contrasigno "
"que essera usate pro acceder al datos post le initio del computator."

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade.h:5
msgctxt "GUI|Passphrase Dialog"
msgid "_Passphrase:"
msgstr "_Phrase de contrasigno:"

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade.h:6
msgctxt "GUI|Passphrase Dialog"
msgid "Con_firm:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade.h:8
msgid ""
"Warning: You won't be able to switch between keyboard layouts (from the "
"default one) when you decrypt your disks after install."
msgstr ""
"Aviso: Le cambiamento del disposition de claviero (del disposition "
"predefinite) non essera possibile si le discos necessita decryptation post "
"le installation."

#: pyanaconda/ui/gui/spokes/lib/refresh.glade.h:1
msgid "RESCAN DISKS"
msgstr "RESCANNAR DISCOS"

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#: pyanaconda/ui/gui/spokes/lib/refresh.glade.h:2
msgctxt "GUI|Refresh Dialog|Rescan"
msgid "_Cancel"
msgstr "_Cancellar"

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#: pyanaconda/ui/gui/spokes/lib/refresh.glade.h:3
msgctxt "GUI|Refresh Dialog|Rescan"
msgid "_OK"
msgstr "_OK"

#: pyanaconda/ui/gui/spokes/lib/refresh.glade.h:4
msgid ""
"You can remove or insert additional disks at this time and press 'Rescan "
"Disks' below for the changes to take effect."
msgstr ""
"Ora on pote remover o inserer dispositivos additional e pressar sur "
"'Rescannar discos' in basso pro effectuar le cambiamentos."

#: pyanaconda/ui/gui/spokes/lib/refresh.glade.h:5
msgid ""
"<b>Warning:</b> All storage changes made using the installer will be lost "
"when you press 'Rescan Disks'."
msgstr ""
"<b>Aviso:</b> Tote le cambiamentos de immagazinage facite per le uso de "
"installator essera perdite quando on pressa sur 'Rescannar discos'."

#: pyanaconda/ui/gui/spokes/lib/refresh.glade.h:6
msgctxt "GUI|Refresh Dialog|Rescan"
msgid "_Rescan Disks"
msgstr "_Rescannar discos"

#: pyanaconda/ui/gui/spokes/lib/refresh.glade.h:7
msgid "Scanning disks.  This may take a moment..."
msgstr "Scannation de discos.  Isto pote prender un momento..."

#: pyanaconda/ui/gui/spokes/lib/refresh.glade.h:8
msgid ""
"You may <a href=\"\">go back to the main menu</a> to complete other\n"
"installation options while this scan completes."
msgstr ""
"Il pote <a href=\"\">retornar al menu principal</a> pro completar altere\n"
"optiones de installation durante le scannation."

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: pyanaconda/ui/gui/spokes/lib/refresh.glade.h:10
msgid "Disk rescan complete."
msgstr "Le operation a rescannar discos se ha completate."

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#: pyanaconda/ui/gui/spokes/lib/resize.glade.h:1
msgctxt "GUI|Reclaim Dialog"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/lib/resize.glade.h:2
msgctxt "GUI|Reclaim Dialog"
msgid "_Reclaim space"
msgstr "_Recuperar spatio"

#: pyanaconda/ui/gui/spokes/lib/resize.glade.h:3
msgid "RECLAIM DISK SPACE"
msgstr "RECUPERAR SPATIO DE DISCO"

#: pyanaconda/ui/gui/spokes/lib/resize.glade.h:4
msgid "Description goes here."
msgstr "Le description es hic."

#: pyanaconda/ui/gui/spokes/lib/resize.glade.h:7
msgid "File System"
msgstr "Systema de files"

#: pyanaconda/ui/gui/spokes/lib/resize.glade.h:8
msgid "Reclaimable Space"
msgstr "Spatio recuperabile"

#: pyanaconda/ui/gui/spokes/lib/resize.glade.h:9
#: pyanaconda/ui/gui/spokes/lib/summary.glade.h:6
msgid "Action"
msgstr "Action"

#: pyanaconda/ui/gui/spokes/lib/resize.glade.h:10
msgctxt "GUI|Reclaim Dialog"
msgid "_Preserve"
msgstr "_Preservar"

#: pyanaconda/ui/gui/spokes/lib/resize.glade.h:11
msgctxt "GUI|Reclaim Dialog"
msgid "_Delete"
msgstr "_Deler"

#: pyanaconda/ui/gui/spokes/lib/resize.glade.h:12
msgctxt "GUI|Reclaim Dialog"
msgid "_Shrink"
msgstr "_Contraher"

#: pyanaconda/ui/gui/spokes/lib/summary.glade.h:1
msgid "SUMMARY OF CHANGES"
msgstr "SUMMARIO DE CAMBIAMENTOS"

#: pyanaconda/ui/gui/spokes/lib/summary.glade.h:2
msgctxt "GUI|Summary Dialog"
msgid "_Cancel & Return to Custom Partitioning"
msgstr "_Cancellar e retornar al partitionamento personalisate"

#: pyanaconda/ui/gui/spokes/lib/summary.glade.h:3
msgctxt "GUI|Summary Dialog"
msgid "_Accept Changes"
msgstr "_Acceptar cambiamentos"

#: pyanaconda/ui/gui/spokes/lib/summary.glade.h:4
msgid ""
"Your customizations will result in the following changes taking effect after "
"you return to the main menu and begin installation:"
msgstr ""
"Le personalisationes effectuara le sequente cambiamentos post le retorno al "
"pagina principal e le comenciamento de installation:"

#: pyanaconda/ui/gui/spokes/lib/summary.glade.h:5
msgid "Order"
msgstr "Ordine"

#: pyanaconda/ui/gui/spokes/lib/summary.glade.h:8
msgid "Device Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade.h:9
msgid "Mount point"
msgstr "Puncto de montage"

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#: pyanaconda/ui/gui/main.glade.h:3
msgctxt "GUI|Main"
msgid "_No"
msgstr "_No"

# translation auto-copied from project anaconda, version f23-branch, document
# po/anaconda
#: pyanaconda/ui/gui/main.glade.h:4
msgctxt "GUI|Main"
msgid "_Yes"
msgstr "_Si"

#: pyanaconda/ui/gui/main.glade.h:5
msgid ""
"Are you sure you wish to quit the\n"
"installation process?"
msgstr ""
"Es tu secur que tu vole quitar le\n"
"processo de installation?"

# translation auto-copied from project ibus, version head, document ibus10
#: pyanaconda/ui/gui/main.glade.h:7
msgid "Quit"
msgstr "Quitar"

#: pyanaconda/ui/gui/hubs/summary.glade.h:1
msgid "INSTALLATION SUMMARY"
msgstr "SUMMARIO DE INSTALLATION"

#: pyanaconda/ui/gui/hubs/summary.glade.h:2
msgctxt "GUI|Summary"
msgid "_Quit"
msgstr "_Quitar"

#: pyanaconda/ui/gui/hubs/summary.glade.h:3
msgctxt "GUI|Summary"
msgid "_Begin Installation"
msgstr "_Comenciar le installation"

#: pyanaconda/ui/gui/hubs/summary.glade.h:4
msgid "We won't touch your disks until you click 'Begin Installation'."
msgstr ""
"Discos non essera afficite usque le clic sur le button 'Comenciar le "
"installation'."

#: pyanaconda/ui/gui/hubs/progress.glade.h:1
msgid "CONFIGURATION"
msgstr "CONFIGURATION"

#: pyanaconda/ui/gui/hubs/progress.glade.h:2
msgid "Preparing to install"
msgstr "Preparation de installation"

#: pyanaconda/ui/gui/hubs/progress.glade.h:3
msgctxt "GUI|Progress"
msgid "_Finish configuration"
msgstr "_Finir le configuration"

#: pyanaconda/ui/gui/hubs/progress.glade.h:4
msgctxt "GUI|Progress"
msgid "_Reboot"
msgstr "_Reinitiar"

#: data/liveinst/liveinst.desktop.in.h:1
msgid "Install to Hard Drive"
msgstr "Installar sur le disco dur"

#: data/liveinst/liveinst.desktop.in.h:2
msgid "Install"
msgstr "Installar"

#: data/liveinst/liveinst.desktop.in.h:3
msgid "Install the live CD to your hard disk"
msgstr "Installar le LiveCD sur le disco dur"

#: /home/rvykydal/anaconda/data/liveinst/console.apps/liveinst:5
msgid "Starting Install to Hard Drive"
msgstr "Initio de installation sur le disco dur"

#: data/liveinst/gnome/fedora-welcome.desktop.in.h:1
#: data/liveinst/gnome/fedora-welcome:65
msgid "Welcome to Fedora"
msgstr "Benvenite a Fedora"

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: anaconda:490
#, python-format
msgid ""
"%(product_name)s requires %(needed_ram)s MB of memory to install, but you "
"only have %(total_ram)s MB on this machine.\n"
msgstr ""
"%(product_name)s require %(needed_ram)s MB de memoria pro le installation, "
"ma iste machina ha solmente %(total_ram)s MB.\n"

#: anaconda:492
#, python-format
msgid ""
"The %(product_name)s graphical installer requires %(needed_ram)s MB of "
"memory, but you only have %(total_ram)s MB\n"
"."
msgstr ""

# translation auto-copied from project Anaconda, version master, document
# anaconda
#: anaconda:495
msgid ""
"\n"
"Press <return> to reboot your system.\n"
msgstr ""
"\n"
"Preme sur <retorno> pro reinitiar le systema.\n"

#: anaconda:497
msgid "Not enough RAM"
msgstr "RAM insufficiente"

#: anaconda:498
msgid ""
" Try the text mode installer by running:\n"
"\n"
"'/usr/bin/liveinst -T'\n"
"\n"
" from a root terminal."
msgstr ""
" Essaya le installator in modo textual con le commando:\n"
"\n"
"'/usr/bin/liveinst -T'\n"
"\n"
" entrate in un terminal de superusator."

#: anaconda:501
msgid " Starting text mode."
msgstr " Initio del modo textual."

#: anaconda:537
msgid "The installation cannot continue and the system will be rebooted"
msgstr ""

#: anaconda:664
msgid "DISPLAY variable not set. Starting text mode."
msgstr "Le variabile DISPLAY non es definite. Le modo textual se initia."

#: anaconda:672
msgid "Graphical installation is not available. Starting text mode."
msgstr "Le installation graphic non es disponibile. Le modo textual se initia."

#: anaconda:680
msgid ""
"Text mode provides a limited set of installation options. It does not offer "
"custom partitioning for full control over the disk layout. Would you like to "
"use VNC mode instead?"
msgstr ""
"Le modo textual provide optiones de installation limitate. Illo non permitte "
"specificar un proprie arrangiamento de partitiones sur le disco. Velle tu "
"usar le modo VNC in su loco?"

#: anaconda:776
#, python-format
msgid "Please ssh install@%s to begin the install."
msgstr "Exeque ssh install@%s pro comenciar le installation."

#: anaconda:778
msgid "Please ssh install@<host> to continue installation."
msgstr "Exeque ssh install@<host> pro continuar le installation."

#: data/liveinst/gnome/fedora-welcome:94
msgid "Try Fedora"
msgstr "Probar Fedora"

#: data/liveinst/gnome/fedora-welcome:110
msgid ""
"You are currently running Fedora from live media.\n"
"You can install Fedora now, or choose \"Install to Hard Drive\" in the "
"Activities Overview at any later time."
msgstr ""
"Actualmente Fedora es lanceate desde un live-medio. \n"
"Il pote installar Fedora ora, o eliger \"Installar sur le disco dur\" in le "
"Resume de activitates plus tarde."

#: data/liveinst/gnome/fedora-welcome:128
msgid ""
"You can choose \"Install to Hard Drive\"\n"
"in the Activities Overview at any later time."
msgstr ""
"Il pote eliger \"Installar sur le disco dur\"\n"
"in le Resume de activitates plus tarde."

#: data/liveinst/gnome/fedora-welcome:131
msgid "Close"
msgstr "Clauder"
